<html>
<head>
<TITLE>Genetics 570</TITLE>
</HEAD>
<body bgcolor="#ccffff">
<CENTER>
<H1>Genome 570</H1><P>
<H2>Phylogenetic Inference</H2>
<H2>Winter, 2010</H2>
</CENTER>
<P>
<h3>Monday, Wednesday, and Friday</h3>
<h3>11:30am-12:20pm</h3>
<h3>S110 Foege Building</h3>
<P>
<UL>
<LI> <A HREF="#news">News </A>
<LI> <A HREF="#catalog">Description from the UW Course Catalog</A>
<LI> <A HREF="syllabus.html">Syllabus of lectures</A>
<LI> <A HREF="#projections">Lecture projections</A>
<LI> <A HREF="#audio">Audio recordings</A>
<LI> <A HREF="#where">Where is the lecture room?</A>
<LI> <A HREF="#text">The Course Text</A>
<LI> <A HREF="#computer">Computer exercises</A>
<LI> <A HREF="#data">Some data sets you can download and use</A>
<LI> <A HREF="http://www.gs.washington.edu/faculty/felsenstein.htm">Who is this guy who is teaching the course, anyway?</A>
<LI> <A HREF="#othercourses">What are some other related courses?</A>
<LI> <A HREF="#internet">What are some Internet resources on evolutionary biology?</A>
<LI> <A HREF="http://evolution.genetics.washington.edu/phylip.html">Where can I get a copy of PHYLIP?</A>
<LI> <A HREF="http://evolution.genetics.washington.edu/phylip/software.html"> What other phylogeny programs are available?</A>
</UL>
<HR> <A name="news">
<H2> News about the course </H2>
<P>
<UL>
<LI> Grades have been submitted.  A point total distribution can be seen
<a href="totals.pdf">here</a>.   If you want your exam back with the
point totals and grade written on it, send me a campus mailing address and
I will send it through campus mails, or
else come to one of my 8 lectures in Genome 541 and I will give it to you.
Genome 541 (Computational Molecular Biology) is in S110 Tuesdays and Thursdays 10:30-11:50, and I will be
there lecturing on 3/30, 4/1, 4/13, 4/15, 4/20, 4/22, 4/27, and 5/4.
Please <i>do</i> get them back from me -- they do me no good sitting in the 
files.  Thanks for being such a good class, willing to ask lots of very
cogent questions.
<P>
<li> I will later post to this list some comments on the exam, some
issues of answers to some ambiguous questions, etc.  But one person is out
of town and (by agreement with me) will not take the exam until the start of
next quarter, so I will
delay those comments until after that.
<p>
<LI> The review session was held Monday morning at 11:00 in S110 and lasted
past 12:30.  I'm sorry not to have posted a notice of it here -- hopefully you
all got email about it in the course email mailing list.  I have put audio
recordings of it with the lecture audio recordings.
<P>
<LI> Homework no. 4 is now posted <a href="homework4.pdf">here</a>.  Due
March 10 (and please do not ask for extensions of time for that one).
<P>
<LI> The final from 2008 is now posted as a PDF <a href="final2008.pdf">here</a>.
Just to give you some idea of what sort of questions I ask.  Of course I won't
ask those questions, but I might ask questions <i>sort of like</i> them.   And I
might even ask one of them again.  Or maybe not ...
<P>
<li> Homework assignment number 3 is now posted <a href="homework3.pdf">
here</a>.  Note that, owing to the delay in our getting to bootstrapping,
I have made it due on Friday, February 26.
<P>
<li> Homework assignment number 2 is now posted <a
href="homework2.pdf">here</a> as a PDF.  It is due on the 3rd, by email of
a PDF or of a Word .DOC file.
<p>
<li> Homework assignment number 1 is now posted <a
href="homework1.pdf">here</a> as a PDF.  Due on the 20th, by email of a PDF.
<P>
<li> Genome 570 has an email mailing list, to which all of its members can post
messages.  To post, email to
<tt>genome570a_wi10<!deathtospam>&nbsp;(at)&nbsp;<!deathtospam>u.washington.edu</tt>
&nbsp;To read archives of past messages, you can go to
    <a
href="https://mailman1.u.washington.edu/mailman/listinfo/genome570a_wi10">
https://mailman1.u.washington.edu/mailman/listinfo/genome570a_wi10</a>
</UL>
<P>
<HR>
<A name="catalog">
<H2> Description from the UW Course Catalog </H2>
<P><B>GENOME 570 Phylogenetic Inference (3) </B><i> Felsenstein </i><BR> Methods for inferring phylogenies (evolutionary trees) - biological assumptions, statistical foundations, and computational methods. A comprehensive introduction for graduate students in the biological sciences to phylogenetic methods using data from molecular sequences, continuous and discrete characters, and gene frequencies. Prerequisite: introductory courses in evolution and in statistics. Offered: alternate years; Sp.
<P>
<HR>
<A NAME="projections">
<H2> Lecture projections</H2>
<P>
Here are the lecture projections.
The ones marked &ldquo;(preliminary)&rdquo; are this year's, but may have
some more material added later.
<ul>
<li> <a href="week1.pdf">Week 1: Parsimony, tree enumeration</a>
<li> <a href="week2.pdf">Week 2: Searching for trees, ancestral states</a>
<li> <a href="week3.pdf">Week 3: Parsimony variants, compatibility, statistics
and parsimony</a>
<li> <a href="week4.pdf">Week 4: History, philosophy, distance methods</a>
<li> <a href="week5.pdf">Week 5: Distance methods, DNA and protein models</a>
<li> <a href="week6.pdf">Week 6: Restriction sites, RAPDs, microsatellites,
likelihood, hidden Markov models</a>
<li> <a href="week7.pdf">Week 7: Bayesian inference, Testing trees, Bootstraps
</a>
<li> <a href="week8.pdf">Week 8: Paired sites tests, gene frequencies,
continuous characters</a>
<li> <a href="week9.pdf">Week 9: Coalescents</a>
<li> <a href="week10.pdf">Week 10: Consensus trees, tree distances</a>
</ul>
<P>
<HR>
<P><A NAME="audio">
<h2>Audio recordings</h2>
<P>
I will be making audio recordings of my lectures.  They will be
posted here in two forms, as WMA files and as MP3 files.  They will
be recorded at a medium quality (to reduce file size) and should be
about 5 Megabytes each.  Watch for them here.  Their names
indicate the date of the lecture: thus the lecture for February 13th
would be files 20100213.WMA and 20100213.mp3.  The 4-digit year is
followed by the 2-digit month and the 2-digit day.
<div align=center>
<table border=1><tr>
<td>
week 1
</td>
<td>
week 2
</td>
<td>
week 3
</td>
<td>
week 4
</td>
<td>
week 5
</td>
<td>
week 6
</td>
</tr>
<tr>
<td>
<a href="20100104.WMA">20100104.WMA</a><BR>
<a href="20100104.mp3">20100104.mp3</a><BR>
<a href="20100106.WMA">20100106.WMA</a><BR>
<a href="20100106.mp3">20100106.mp3</a><BR>
<a href="20100108.WMA">20100108.WMA</a><BR>
<a href="20100108.mp3">20100108.mp3</a>
</td>
<td>
<a href="20100111.WMA">20100111.WMA</a><BR>
<a href="20100111.mp3">20100111.mp3</a><BR>
<a href="20100113.WMA">20100113.WMA</a><BR>
<a href="20100113.mp3">20100113.mp3</a><br>
<a href="20100115.WMA">20100115.WMA</a><BR>
<a href="20100115.mp3">20100115.mp3</a>
</td>
<td>
<a href="20100120.WMA">20100120.WMA</a><BR>
<a href="20100120.mp3">20100120.mp3</a><BR>
<a href="20100122.WMA">20100122.WMA</a><BR>
<a href="20100122.mp3">20100122.mp3</a>
</td>
<td>
<a href="20100125.WMA">20100125.WMA</a><BR>
<a href="20100125.mp3">20100125.mp3</a><BR>
<a href="20100127.WMA">20100127.WMA</a><BR>
<a href="20100127.mp3">20100127.mp3</a><BR>
<a href="20100129.WMA">20100129.WMA</a><BR>
<a href="20100129.mp3">20100129.mp3</a>
</td>
<td>
<a href="20100201.WMA">20100201.WMA</a><BR>
<a href="20100201.mp3">20100201.mp3</a><BR>
<a href="20100203.WMA">20100203.WMA</a><BR>
<a href="20100203.mp3">20100203.mp3</a><BR>
<a href="20100205.WMA">20100205.WMA</a><BR>
<a href="20100205.mp3">20100205.mp3</a>
</td>
<td>
<a href="20100208.WMA">20100208.WMA</a><BR>
<a href="20100208.mp3">20100208.mp3</a><BR>
<a href="20100210.WMA">20100210.WMA</a><BR>
<a href="20100210.mp3">20100210.mp3</a><BR>
<a href="20100212.WMA">20100212.WMA</a><BR>
<a href="20100212.mp3">20100212.mp3</a>
</td>
</tr>
<tr>
<td>
week 7
</td>
<td>
week 8
</td>
<td>
week 9
</td>
<td>
week 10
</td>
</tr>
<tr>
<td>
<a href="20100217.WMA">20100217.WMA</a><BR>
<a href="20100217.mp3">20100217.mp3</a><br>
<a href="20100219.WMA">20100219.WMA</a><BR>
<a href="20100219.mp3">20100219.mp3</a>
</td>
<td>
<a href="20100222.WMA">20100222.WMA</a><BR>
<a href="20100222.mp3">20100222.mp3</a><BR>
<a href="20100224.WMA">20100224.WMA</a><BR>
<a href="20100224.mp3">20100224.mp3</a><BR>
<a href="20100226.WMA">20100226.WMA</a><BR>
<a href="20100226.mp3">20100226.mp3</a>
</td>
<td>
<a href="20100301.WMA">20100301.WMA</a><BR>
<a href="20100301.mp3">20100301.mp3</a><BR>
<a href="20100303.WMA">20100303.WMA</a><BR>
<a href="20100303.mp3">20100303.mp3</a><BR>
<font size=-1>
Ouch! forgot to record the lecture of 3/05!<br>
(see Chapter 26 of the book and maybe<br>
also <a href="http://evolution.gs.washington.edu/gs541/2010/20100427.WMA">this</a>
(<a href="http://evolution.gs.washington.edu/gs541/2010/20100427.mp3">this</a>)
recorded lecture from<br>
<! Genome 453 last autumn). >
Genome 541 in Spring.
</font>
</td>
<td>
<a href="20100308.WMA">20100308.WMA</a><BR>
<a href="20100308.mp3">20100308.mp3</a><BR>
<a href="20100310.WMA">20100310.WMA</a><BR>
<a href="20100310.mp3">20100310.mp3</a><br>
<a href="20100312.WMA">20100312.WMA</a><BR>
<a href="20100312.mp3">20100312.mp3</a>
</td>
<td>
<a href="ReviewSession.WMA">Review Session (WMA)</a></br>
<a href="ReviewSession.mp3">Review Session (MP3)</a>
</td>
</tr>
</table>
</div>
<P>
<HR>
<P><A NAME="where">
<h2>Where is the lecture room?</h2>
<P>
From upper campus, come across the 
pedestrian bridge (over Pacific Avenue) next to Kincaid Hall (the bridge 
nearest 15th avenue).  Walk forward past Hitchcock Hall on the right, 
descending to ground level.  Then turn right on the walkway that comes out of 
the Health Sciences Building.  Foege (&ldquo;FAY-gee&rdquo;) Building is ahead 
of you, and the S (South) wing of it is the left half, the downhill half.  
Enter it at the door which is on the left side of that sidewalk as you reach 
the building.  S110 is the first room on your left as you go along this 1st 
floor corridor.
<P>
Alternatively, if you come down 15th avenue, Foege Building is on
your left after you cross Pacific Avenue.  To reach the S (South) wing,
go halfway along the outside of the building and enter through the
opening in the building, entering the part of the building closer to
the bay.
<P>
<font size=+2>
<a href="http://www.washington.edu/maps/?l=GNOM">Here is a campus map showing the location.</a>
</font>
<P>
Oh yes, and who is the building named after?  Someone living:
<a href="http://en.wikipedia.org/wiki/William_Foege">This guy.</a>  Not bad!

<p>
<HR>
<A name="text">
<H2> The course text and how to get it</H2>
<P>
The course text is my book,
<STRONG>Inferring Phylogenies</STRONG>, published by Sinauer Associates.  The 
University Book Store (South Campus Center branch) will have copies.  It 
can also be ordered from Sinauer Associates, from Amazon, from Barnes and
Noble, etc.  The first printing 
of the book has many typos.  A
<a href="http://evolution.gs.washington.edu/book/typos.html">web page listing the typos</a>
is available.  Many of these were corrected in the second printing, 
and some more in the third printing.  If there is a box on the back cover with 
reviews of the book, that is the second or third edition.
<P>
<HR>
<A name="computer">
<h2>Computer exercises</h2>
<P>
There will be several computer exercises that you will be asked to do.
These will use either data sets accessible here, or data of your own
choosing.
The exercises can use my own PHYLIP programs or any other phylogeny program you want
to use.  For an extensive list of available programs, with links to their
web pages, look
<div align=center>
<table border=1><tr><td>
<font size=+2><a href="http://evolution.gs.washington.edu/phylip/software.html">here</a></font>
</td></tr></table>
</div>
The results of the exercises are to be handed in as homeworks.
<P>
Here are the homeworks assigned (so far):
<ul>
<li> <a href="homework1.pdf">Homework 1</a>
<li> <a href="homework2.pdf">Homework 2</a>
<li> <a href="homework3.pdf">Homework 3</a>
<li> <a href="homework4.pdf">Homework 4</a>
</ul>
<P>
<HR>
<A name="data">
<H2>Some data sets you can download and use</H2>
<P>
Almost all the data sets used in the book can be downloaded from its
<a href="http://evolution.gs.washington.edu/book/datasets.html">data sets web site</a>.
<P>
These additional data sets are so that we can discuss common examples when
people try
out various programs.  They are in PHYLIP formats, but these can often be
read by other programs such as PAUP and MacClade.  These data sets can be
downloaded by clicking on the appropriate words.  If clicking on
them happens to display the file rather than open a window that asks you
where to put the file, you may be able to get your browser to save it by
using a "<EM>Save As</EM>" function (on most browsers that's in the Files menu).
<UL>
<LI> A 15-species, 40-character discrete-characters (0/1) data set
of fossil elephants by Pascal Tassy and Pierre Darlu.  From a paper in
<EM>Geobios</EM> <STRONG>19:</STRONG> 587-600  1986.    <A HREF="http://evolution.gs.washington.edu/gs570/2008/elephants.dat">Click here to download.</A>
<LI> A 16-species, 1992-site DNA data set from the paper by Clint
Turbeville and others:<BR>
Turbeville. J. McC., Schulz, J .R. and R. A. Raff.
1994. Deuterostome phylogeny and the sister group of the chordates: evidence from
molecules and morphology.  <EM>Molecular Biology and Evolution</EM> <STRONG>11:</STRONG> 648-655.<BR>
 Alignment is by them.  These are chordates, protochordates, echinoderms,
thisachordates, and thatachordates.  Heavy-duty industrial-strength data. 
These sequences are in interleaved format.
<A HREF="http://evolution.gs.washington.edu/gs570/2008/turbeville.dna">Click here to download.</A>
<LI> A 76-species protein sequence data set for Transcription Elongation
Factor Alpha (EFTU-alpha), alignment by the Protein Information Resource
of the National Biomedical Research Foundation.
<A HREF="http://evolution.gs.washington.edu/gs570/2008/eftualpha.prt">Click here to download.</A>
Some species are represented more than once, and some by fragments.
</ul>
<P>
Here are some other sources of aligned sequence data:
<ul>
<LI> A database of aligned mammalian gene sequences, OrthoMAM, is available
<a href="http://www.orthomam.univ-montp2.fr/orthomam/html/index.php?article=Request">here</a>.
Just click on the Submit button.  To see which protein each locus is, click on
the identifier field.
You can retrieve amino acid alignments in FASTA format.  You can copy and then paste the
sequences into <a href="http://www-bimas.cit.nih.gov/molbio/readseq/">this sequence
conversion web page</a> and choose "PHYLIP" format from its menu.
<LI> The <a href="http://www.pfam.org">Pfam</a> database of aligned
protein sequences is hosted by the European Bioinformatics institute.
A particularly easy way to use it is to use the <a
href="http://www.ebi.ac.uk/goldman-srv/pandit/">Pandit</a> web site which
makes Pfam alignments available in a format nearly compatible with
PHYLIP (having ten-character names followed by sequences).  This can be
converted to PHYLIP format by adding a first line with the number of
species followed by the number of sites (the two numbers separated by
one or more blanks).
</UL>
I will be putting up some more datasets and web sites of aligned
sequences.
<hr>
<a name="othercourses"> <strong> What are some other related courses?</strong>
</a><p>
</p><dl><dt>Biology 354 <em>(Foundations in Evolution and Systematics)</em>
<dd><a name="othercourses"> The main evolution course at the University, taught
twice yearly (Winter and Spring).  Texts are &ldquo;Evolution&rdquo; by
Barton et al. and Weiner's &ldquo;The Beak of the Finch&rdquo;.
This year's course is being taught by Peter Ward in the
Winter quarter, and is also being taught by Gregory Wilson in the
Spring Quarter. Ward, a paleontologist, is noted for his work on mass
extinctions and has written many books on evolution for the general public.
Wilson, who works on fossil mammals, is also a paleontologist.<br>
<DT> Genome 453 <em>(Evolutionary Genetics)</em>
<DD> This is an
undergraduate-level course on evolution and genetics.  Given every
Autumn, it overlaps somewhat with Biology 354 but
covers the genetic aspects of evolution with less emphasis on
ecology and paleontology than the first, less emphasis on molecular
evolution than the second.  In even-numbered years the course is given by
Mary Kuhner, in odd-numbered years by me.
<i>It should be understood that this is basically an undergraduate course.</i>
<dt> Biology 550 <em>(Evolution and Systematics Seminar)</em>
<dd> Weekly seminar with readings, presentations, and lots of discussion, the
main seminar for biology graduate students to discuss evolutionary biology.
Run by Toby Bradshaw, who keeps interest stirred up.  Fridays at 1:30,
this quarter in Hitchcock 320.  Every quarter during the regular academic year.
<dt> Genome 562 <em>(Population Genetics)</em>
<dd><a name="othercourses"> Now given every other year, this is the graduate theoretical
evolutionary genetics course that I give.  Lots of equations, though
mostly at a low mathematical level.  No pictures of cute furry animals.
Next time it's given will be Winter, 2011.  Text: my own extensive
<a href="http://evolution.gs.washington.edu/pgbook/">lecture notes</a>,
downloaded
as a PDF for free or sold as printed copies in the course inexpensively
(no royalty is paid to me).
</dl>
<hr>
<a name="internet">
<strong> What are some Internet resources on evolutionary biology?</strong>
</a><p>
There are many:
<P>
<a name="internet"><strong>Electronic journals</strong>
</a></p><p>
There is of course, the professional literature in evolutionary
biology.
Some of these journals (links given below) are available in
electronic versions for UW people.  If these links don't give you access
you should use the Electronic Journals links in the University Library site,
and type in the name of the journal.  If you are at another institution,
you may be able to access a different run of years of each journal.
Here are some direct UW links to the
leading journals covering evolution:
</a></p><ul>
<li><a name="internet"> </a><a href="http://mbe.oupjournals.org/">Molecular
Biology and Evolution</a>
<li> Systematic Biology
   <ul>
     <li> <a
href="http://barbarina.catchword.com/vl=4772093/cl=13/nw=1/rpsv/catchword/tandf/10635157/contp1-1.htm">1998-present</a>
     <li> <a href="http://www.jstor.org/journals/10635157.html">1992-2006</a>

     <li> <a href="http://www.jstor.org/journals/00397989.html">1952-1991 as
<em>Systematic Zoology</em></a>
   </ul>
<li> Evolution
   <ul>
   <li> <a
href="http://www.bioone.org/bioone/?request=get-journals-list&amp;issn=0014-3820">2000-present</a>
   <li> <a href="http://www.jstor.org/journals/00143820.html">1947-2006</a>

   </ul>
<li> <a
href="http://link.springer-ny.com/link/service/journals/00239/tocs.html">Journal
of Molecular Evolution (1997-present)</a>
</li><li> <a
href="http://www.sciencedirect.com/science?_ob=JournalURL&_cdi=6963&_auth=y&_acct=C000029718&_version=1&_urlVersion=0">Molecular
Phylogenetics and Evolution (1993-present) </a>
<li> American Naturalist
<ul>
<li> <a href="http://www.jstor.org/journals/00030147.html">1867-2004</a>
<li> <a href="http://www.journals.uchicago.edu/AN/">1997-present</a>
</ul>
<li> <a href="http://www.biomedcentral.com/bmcevolbiol/">BMC Evolutionary
Biology</a> (Open Access) 2001-present

</DL>
</ul>
<p>
<STRONG> Blogs and Newsgroups </STRONG>
<P>
There are blogs, mostly creation / evolution debating, and some Usenet
newsgroups.  The latter are accessible through Google Groups.  Blogs have
taken over from newsgroups, which are basically nearly dead now.  I do not
have enough understanding of Facebook to know what is available there.
<P>
Some brief descriptions of some of the major ones covering evolution:
<DL>
<DT> <a href="http://treethinkers.blogspot.com">Dechronization</a> (Blog)
<DD> Named after
a science fiction novel written by the famous evolutionary biologist George
Gaylord Simpson, this is a technically oriented blog run by about 6
young phylogeny-oriented evolutionary biologists.  Aside from occasional
natural history it is mostly devoted to phylogenetic methods.
<DT> <a href="http://www.pandasthumb.org">Panda's Thumb</a> (Blog)
<DD> Blog defending evolutionary biology against creationists and advocates of
Intelligent Design, run by Reed Cartwright, a phylogeny postdoc.
<DT> <a href="http://www.uncommondescent.com">Uncommon Descent</a> (Blog)
<DD> Blog centered around William Dembski and Denyse O'Leary, critics of
evolutionary biology and advocates of Intelligent Design.
<DT> <a
href="http://groups.google.com/group/sci.bio.evolution/">sci.bio.evolution</a>

(Usenet newsgroup)
<DD> Moderated by Josh Hayes, formerly of our own Center for Quantitative
Sciences,
who should get some sort of award for putting up with a lot of nonsense.
I think it was intended as a forum for discussion among researchers, but has
tended to be filled with postings by amateurs, including some high-volume
cranks.  In among them are some people with serious knowledge. Not intended for
evolution/creation debates: Josh screens these out.  Lately traffic has
declined, as newsgroups are disappearing.
<DT> <a href="http://groups.google.com/group/talk.origins">talk.origins</a>
(Usenet newsgroup)
<DD> The arena for endless debate between creationists and others, with
frequent digressions into theology (or maybe it's for discussion of theology
with occasional digressions into biology).  Extremely high noise to signal
ratio.
When a decisive point is made, the opponent changes the subject or just
refuses to respond.  The pro-evolution posters generally do a good job.
</DL>
<P>
<strong>Web Pages</strong>
</p>
<ul>
<li> The amazing <a href="http://tolweb.org/tree/phylogeny.html"> Tree of Life
</a>, a phylogeny of all life, in the
making.  Trees, figures, descriptions and references at a solid professional
level, each page done by a major systematist working on that group.
<li> <a href="http://www.treebase.org/treebase/">TreeBASE</a>, a database of
evolutionary trees (phylogenies)
from the scientific literature.  Weighted towards flowering plants, and
badly suffers being only sporadically updated.

<li> My own <a href="http://evolution.gs.washington.edu/phylip.html">
PHYLIP </a> free package of computer programs for inferring phylogenies.
Includes some web pages on all possible phylogeny programs and how to get
them.
<li> But maybe you don't want to be limited to just my programs?  OK,
try this listing which I maintain of all known
<a href="http://evolution.gs.washington.edu/phylip/software.html">Phylogeny
Programs</a>.  As of January 2010 there are 385 programs and 53 web servers
listed.  Many of the programs are freely downloadable.
<li> Want some DNA or protein sequences?  How about getting them from the
international database?  Try the <a
href="http://www.ncbi.nlm.nih.gov/Database/index.html">Web pages of the
databases</a> at the
NCBI, the National Center for Biotechnology Information of the National
Library of Medicine, in Bethesda, Maryland.  Not for the faint-hearted.
<li> The <a href="http://www.talkorigins.org/origins/faqs.html">
talk.origins Evolution FAQs</a> (Frequently Asked Questions).
<li> The <a href="http://mcb.harvard.edu/BioLinks/Evolution.html">Harvard
Biopages evolution page</a>, with lots of links to other
evolution resources.
</UL>
<P>
<hr>
This page maintained fitfully by Joe Felsenstein
</body></html>
