<html>
<head>
<TITLE>Genetics 570</TITLE>
</HEAD>
<body bgcolor="#ccffff">
<H1>Genome 570</H1><P>
<H2>Phylogenetic Inference</H2>
<H2>Winter, 2016</H2>
</CENTER>
<P>
<h3>Monday, Wednesday, and Friday</h3>
<h3>11:30am-12:20pm</h3>
<h3>S110 Foege Building</h3>
<P>
<UL>
<LI> <A HREF="#news">News </A>
<LI> <A HREF="#catalog">Description from the UW Course Catalog</A>
<LI> <A HREF="#where">Where is the lecture room?</A>
<LI> <A HREF="#mailist">Course email mailing list</A>
<LI> <A HREF="syllabus.html">Syllabus of lectures</A>
<LI> <A HREF="#materials">Lecture projections and audio recordings</A>
<LI> <A HREF="#text">The Course Text</A>
<LI> <A HREF="#reserve">Books on Reserve</A>
<LI> <A HREF="#computer">Computer exercises (homeworks)</A>
<LI> <A HREF="#data">Some data sets you can download and use</A>
<LI> <A HREF="http://evolution.gs.washington.edu/felsenstein.html">Who is this guy who is teaching the course, anyway?</A>
<LI> <A HREF="#othercourses">What are some other related courses?</A>
<LI> <A HREF="#internet">What are some Internet resources on evolutionary biology?</A>
<LI> <A HREF="http://evolution.genetics.washington.edu/phylip.html">Where can I get a copy of PHYLIP?</A>
<LI> <A HREF="http://evolution.genetics.washington.edu/phylip/software.html"> What other phylogeny programs are available?</A>
</UL>
<HR> <A name="news">
<H2> News about the course </H2>
<P>
<UL>
<li> The exam is now graded, the course point totals computed, and the grades
assigned and submitted to the UW's grading system.  I am glad you all survived
my overly-difficult exam.  Be assured that in grading it I was aware of this
and compensated for it.  A histogram of the course point totals will be found
<a href="totals.pdf">here</a>.  I am also starting to mail the exams, and any
leftover homeworks, back to you.  On each exam I have written the exam total
score, the course total score, and the grade assigned.  Thanks to all of you
for being a very good class, and for asking so many good questions.
<li> The Review Session will be held on Tuesday, the 15th of March (that's
right -- the ominous Ides of March) from 3:00-5:00pm, in the classroom.  The
session will be recorded and the audio recording posted immediately
afterwards.
<li> <a href="hw4/homework4.pdf">Homework no. 4</a> has been assigned, due at the end of Wednesday, the 2nd of March.
<li> <a href="hw3/homework3.pdf">Homework no. 3</a> has been assigned, due at the end of Friday, the 19th of February.
<li> <a href="hw2/homework2.pdf">Homework no. 2</a> has been assigned, due at the end of Friday, the 5th of February.
<li> <a href="hw1/homework1.pdf">Homework no. 1</a> has been assigned, due Wednesday, the 20th of January.
</UL>
<P>
<HR>
<A name="catalog">
<H2> Description from the UW Course Catalog </H2>
<P><B>GENOME 570 Phylogenetic Inference (3) </B><i> Felsenstein </i><BR> Methods for inferring phylogenies (evolutionary trees) - biological assumptions, statistical foundations, and computational methods. A comprehensive introduction for graduate students in the biological sciences to phylogenetic methods using data from molecular sequences, continuous and discrete characters, and gene frequencies. Prerequisite: introductory courses in evolution and in statistics. Offered: alternate years; Sp.
<P>
<HR>
<P><A NAME="where">
<h2>Where is the lecture room?</h2>
<P>
<b>Directions to the lecture room:</b>The building used to be reachable
from upper campus using the overpass near Kincaid Hall.  But alas,
that route is impossible owing to construction.
From upper campus,
come down 15th avenue. Foege Building is on
your left after you cross Pacific Avenue.  To reach the S (South) wing,
go halfway along the outside of the building and enter through the
opening in the building, entering the part of the building closer to
the bay.
Enter it at the door which is to the right as
you pass through the opening in the building.  S110 is the first
room on your left as you go along this 1st floor corridor.
<P>
<font size=+2>
<a href="http://www.washington.edu/maps/?l=GNOM">Here is a campus map showing the location.</a>
</font>
<P>
Oh yes, and who is the building named after?  Someone living:
<a href="http://en.wikipedia.org/wiki/William_Foege">This guy.</a>  Not bad!

<P>
<HR>
<A NAME="mailist">
<h2>Course email mailing list</h2>
<P>
The course now has an email mailing list, to which all registered students have
been subscribed.  You can view past mailings
<a href="http://mailman.u.washington.edu/mailman/listinfo/genome570a_wi16">here</a>).  To send mail to the whole mailing
list, email to &nbsp;<tt>genome570a_wi16@u.washington.edu"</tt>&nbsp; Everyone
on the list can email to the whole list.  I will also post announcements about
the course and answer questions there.  If people email me questions at my
own email address about the
course, if they are of general interest I will often use the mailing list to answer them, after anonymizing who
asked the question.
<P>
<HR>
<A NAME="materials">
<h2>Lecture projections and audio recordings</h2>
<P>
Here are the lecture projections.
The ones marked &ldquo;(old)&rdquo; are from the 2014 course; they will be
replaced by the lecture dates in 2016 by ones which are
similar but which will have some more material added.
<p>
I will be making audio recordings of my lectures.  They will be
posted here in two forms, as WMA files and as MP3 files.  They will
be recorded at a medium quality (to reduce file size) and should be
about 10-12 Megabytes each.  Watch for them here.  Their names
indicate the date of the lecture: thus the lecture for February 13th
would be files 20160213.WMA and 20160213.mp3.  The 4-digit year is
followed by the 2-digit month and the 2-digit day.
<p>
I will put the audio files here as soon after the lecture as I can.  In the
meantime, if you want to hear the corresponding lectures from 2014, you
will find them at <a href="http://evolution.gs.washington.edu/gs570/2014/">the
2014 course web page</a>.
<p>
The audio recordings are organized according to the topic being covered at
the beginning of the talk (so to find a given topic you may need to start
in the last recording of the previous group).
<div align=center>
<table border=1><tr>
<tr><td><b>Week</b></td><td><b>&nbsp;&nbsp;&nbsp;Lecture
projections (PDFs)</b></td><td><b>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Lecture
recordings&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<br>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;(WMA
format)</b>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</td><td>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<b>Lecture
recordings&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<br>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;(MP3 format)</b>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</td></tr>
<tr><td>Week 1</td><td>&nbsp;&nbsp;&nbsp;<a href="week1.pdf">Parsimony, tree
enumeration</a> </td><td align=center>
<a href="audio/20160104.WMA">20160104.WMA</a><br>
<a href="audio/20160106.WMA">20160106.WMA</a><br>
<a href="audio/20160108.WMA">20160108.WMA</a><br>
</td>
<td align=center>
<a href="audio/20160104.mp3">20160104.mp3</a><br>
<a href="audio/20160106.mp3">20160106.mp3</a><br>
<a href="audio/20160108.mp3">20160108.mp3</a><br>
</td>

</tr>
<tr><td>Week 2</td><td>&nbsp;&nbsp;&nbsp;<a href="week2.pdf">Searching for trees, ancestral
states, parsimony variants</a></td><td align=center>
<a href="audio/20160111.WMA">20160111.WMA</a><br>
<a href="audio/20160113.WMA">20160113.WMA</a><br>
<a href="audio/20160115.WMA">20160115.WMA</a><br>
 </td>
<td align=center>
<a href="audio/20160111.mp3">20160111.mp3</a><br>
<a href="audio/20160113.mp3">20160113.mp3</a><br>
<a href="audio/20160115.mp3">20160115.mp3</a><br>
 </td></tr>
<tr><td>Week 3</td><td>&nbsp;&nbsp;&nbsp;<a href="week3.pdf">Character
weighting, compatibility </a> </td><td align=center>
<a href="audio/20160120.WMA">20160120.WMA</a><br>
<a href="audio/20160122.WMA">20160122.WMA</a><br>
</td><td align=center>
<a href="audio/20160120.mp3">20160120.mp3</a><br>
<a href="audio/20160122.mp3">20160122.mp3</a><br>
</td></tr>
<tr><td>Week 4 </td><td>&nbsp;&nbsp;&nbsp;<a href="week4.pdf">Statistics and
parsimony. History and philosophy
</a> </td><td align=center>
<a href="audio/20160125.WMA">20160125.WMA</a><br>
<a href="audio/20160127.WMA">20160127.WMA</a><br>
<a href="audio/20160129.WMA">20160129.WMA</a><br>
</td><td align=center>
<a href="audio/20160125.mp3">20160125.mp3</a><br>
<a href="audio/20160127.mp3">20160127.mp3</a><br>
<a href="audio/20160129.mp3">20160129.mp3</a><br>
</td></tr>
<tr><td>Week 5</td><td>&nbsp;&nbsp;&nbsp;<a href="week5.pdf">Distance methods, DNA models</a> </td>
<td align=center>
<a href="audio/20160201.WMA">20160201.WMA</a><br>
<a href="audio/20160203.WMA">20160203.WMA</a><br>
<a href="audio/20160205.WMA">20160205.WMA</a><br>
 </td>
<td align=center>
<a href="audio/20160201.mp3">20160201.mp3</a><br>
<a href="audio/20160203.mp3">20160203.mp3</a><br>
<a href="audio/20160205.mp3">20160205.mp3</a><br>
</td></tr>
<tr><td>Week 6</td><td>&nbsp;&nbsp;&nbsp;<a href="week6.pdf">Protein and codon models,
Likelihood methods, rate variation</a> </td>
<td align=center>
<a href="audio/20160208.WMA">20160208.WMA</a><br>
<a href="audio/20160210.WMA">20160210.WMA</a><br>
<a href="audio/20160212.WMA">20160212.WMA</a><br>
</td>
<td align=center>
<a href="audio/20160208.mp3">20160208.mp3</a><br>
<a href="audio/20160210.mp3">20160210.mp3</a><br>
<a href="audio/20160212.mp3">20160212.mp3</a><br>
</td></tr>
<tr><td>Week 7</td><td>&nbsp;&nbsp;&nbsp;<a href="week7.pdf">Bayesian
inference</a> </td><td align=center>
<a href="audio/20160217.WMA">20160217.WMA</a><br>
<a href="audio/20160219.WMA">20160219.WMA</a><br>
</td><td
align=center>
<a href="audio/20160217.mp3">20160217.mp3</a><br>
<a href="audio/20160219.mp3">20160219.mp3</a><br>
</td></tr>
</a>
<tr><td>Week 8</td><td>&nbsp;&nbsp;&nbsp;<a href="week8.pdf">Testing trees, bootstraps, jackknifes, gene frequencies</a>
 </td><td align=center>
<a href="audio/20160222.WMA">20160222.WMA</a><br>
<a href="audio/20160224.WMA">20160224.WMA</a><br>
<a href="audio/20160226.WMA">20160226.WMA</a><br>
</td><td align=center>
<a href="audio/20160222.mp3">20160222.mp3</a><br>
<a href="audio/20160224.mp3">20160224.mp3</a><br>
<a href="audio/20160226.mp3">20160226.mp3</a><br>
</td></tr>
<tr><td>Week 9</td><td>&nbsp;&nbsp;&nbsp;<a href="week9.pdf">Covariances, comparative methods, coalescents</a> </td><td align=center>
<a href="audio/20160229.WMA">20160229.WMA (*)</a><br>
<a href="audio/20160302.WMA">20160302.WMA (*)</a><br>
<a href="audio/20160304.WMA">20160304.WMA</a><br>
</td><td align=center>
<a href="audio/20160229.mp3">20160229.mp3 (*)</a><br>
<a href="audio/20160302.mp3">20160302.mp3 (*)</a><br>
<a href="audio/20160304.mp3">20160304.mp3</a><br>
</td></tr>
<tr><td>Week 10</td><td>&nbsp;&nbsp;&nbsp;<a href="week10.pdf">Likelihoods on
coalescents, consensus trees, etc. </a> </td><td align=center>
<a href="audio/20160307.WMA">20160307.WMA</a><br>
<a href="audio/20160309.WMA">20160309.WMA</a><br>
<a href="audio/20160311.WMA">20160311.WMA</a><br>
</td><td align=center>
<a href="audio/20160307.mp3">20160307.mp3</a><br>
<a href="audio/20160309.mp3">20160309.mp3</a><br>
<a href="audio/20160311.mp3">20160311.mp3</a><br>
</td></tr>
<tr><td>Review session</td><td>(Given 15 March)</td><td align=center>
<a href="audio/Review.WMA">Review.WMA</a>
</td><td
align=center>
<a href="audio/Review.mp3">Review.mp3</a>
</td></tr>
</table>
<P>
(*) on 29 February 2016 the sound recorder ran out of space after
33 minutes of recording, just as the section on Quantitative Characters was
starting.  The reason for the problem was not understood until 15 minutes
into the next lecture, the one for 2 March 2016. For similar material covering
the gap, see the 2014 web page (<a
href="http://evolution.gs.washington.edu/gs570/2014/#materials">here</a>) and
the recording of the lecture
of 3 March 2014 from about time 13 minutes and 40 seconds to almost the end of
that lecture.
</div>
<p>
<HR>
<A name="text">
<H2> The course text and how to get it</H2>
<P>
The course text is my book,
<STRONG>Inferring Phylogenies</STRONG>, published by Sinauer Associates.  The 
University Book Store (South Campus Center branch) will have copies.  It 
can also be ordered from Sinauer Associates, from Amazon, from Barnes and
Noble, etc.  The first printing 
of the book has many typos.  A
<a href="http://evolution.gs.washington.edu/book/typos.html">web page listing the typos</a>
is available.  Many of these were corrected in the second printing, 
and some more in the third printing.  If there is a box on the back cover with 
reviews of the book, that is the second, third or fourth printing.
<P>
<HR>
<A name="reserve">
<h2> Books on Reserve </h2>
<P>
Two other textbooks have been placed <a
href="http://alliance-primo.hosted.exlibrisgroup.com/primo_library/libweb/action/search.do?vid=UW&vl%28165135230UI0%29=any&group=GUEST&onCampus=true&displayMode=full&query=&fn=search&srt=rank&indx=1&mode=Basic&ct=search&tab=cr&dscnt=0&vl%28freeText0%29=Genome+570">on
24-hour reserve at the Health Sciences
Library</a> (which is at the east end of the 3rd floor of the T wing of Health Sciences):
<ul>
<li> <i>Computational Molecular Evolution</i> by Ziheng Yang
<li> <i>Phylogenetics</i> by Charles Semple and Michael Steel
</ul>
One other textbook that I requested be placed there turned out to be on
"closed reserve" at the Odegaard Undergraduate Library (OUGL), so it can be
read there:
<ul>
<li> <i>Tree Thinking: An Introduction to Phylogenetic Biology</i> by David Baum and Stacey Smith
</ul>
These are well worth looking at.  Yang's book is a well-organized and carefully
thought-out introduction to molecular phylogenetics.  Semple and Steel's book
is an introduction from the point of view of mathematicians, with lots of
definitions, theorems, and proofs.  Baum and Smith's book is intended as an
accessible introduction at an undergraduate level, and it is worth seeing how
they accomplish that.
<p>
Note that Yang's book is also available through the UW Library web pages, in an
electronic book format.
<p>

<P>
<HR>
<A name="computer">
<h2>Computer exercises (homeworks)</h2>
<P>
There will be several computer exercises that you will be asked to do.
These will use either data sets accessible here, or data of your own
choosing.
The exercises can use my own PHYLIP programs or any other phylogeny program you want
to use.  For an extensive list of available programs, with links to their
web pages, look
<div align=center>
<table border=1><tr><td>
<font size=+2><a href="http://evolution.gs.washington.edu/phylip/software.html">here</a></font>
</td></tr></table>
</div>
The results of the exercises are to be handed in as homeworks.
<P>
Here are the homeworks assigned (so far):
<ul>
<li> <a href="hw1/homework1.pdf">Homework 1</a>, due Wednesday, the 20th of January.
<li> <a href="hw2/homework2.pdf">Homework 2</a>, due at the
end of Friday, the 5th of February.
<li> <a href="hw3/homework3.pdf">Homework 3</a>, due at the end of Friday, the 19th of February.
<li> <a href="hw4/homework4.pdf">Homework 4</a>, due at the end of Wednesday, the
2nd of March.
</ul>
<P>
<HR>
<A name="data">
<H2>Some data sets you can download and use</H2>
<P>
Almost all the data sets used in the book can be downloaded from its
<a href="http://evolution.gs.washington.edu/book/datasets.html">data sets web site</a>.
<P>
These additional data sets are so that we can discuss common examples when
people try
out various programs.  They are in PHYLIP formats, but these can often be
read by other programs such as PAUP and MacClade.  These data sets can be
downloaded by clicking on the appropriate words.  If clicking on
them happens to display the file rather than open a window that asks you
where to put the file, you may be able to get your browser to save it by
using a "<EM>Save As</EM>" function (on most browsers that's in the File menu).
<UL>
<li> The 14-species primates data set that I have used a number of times as an
example.  It includes mouse and cow (bovine) as outgroups.  This data set was
assembled by Dr. Masami Hasegawa of the Institute of Statistical Mathematics
in Tokyo, from sequencing done by Kenji Hayasaka, Takashi Gojobori, and Satoshi
Horai (<i>Molecular Biology and Evolution</i> <b>5:</b> 626-644, 1988).
The sequences were DNA in or near the control region of the mitochondrion.
The 232 base positions selected by Dr. Hasegawa are from the control region,
 plus third codon positions in adjacent protein-coding mitochondrial genes.
He avoided 1st and 2nd codon postions in an attempt to achieve similar
evolutionary rates at all sites. <a href="primates.dna">Click here to 
download.</a>
<LI> A 15-species, 40-character discrete-characters (0/1) data set
of fossil elephants by Pascal Tassy and Pierre Darlu.  From a paper in
<EM>Geobios</EM> <STRONG>19:</STRONG> 587-600  1986.    <A HREF="http://evolution.gs.washington.edu/gs570/2008/elephants.dat">Click here to download.</A>
<LI> A 16-species, 1992-site DNA data set from the paper by Clint
Turbeville and others:<BR>
Turbeville. J. McC., Schulz, J .R. and R. A. Raff.
1994. Deuterostome phylogeny and the sister group of the chordates: evidence from
molecules and morphology.  <EM>Molecular Biology and Evolution</EM> <STRONG>11:</STRONG> 648-655.<BR>
 Alignment is by them.  These are chordates, protochordates, echinoderms,
thisachordates, and thatachordates.  Heavy-duty industrial-strength data. 
These sequences are in interleaved format.
<A HREF="data/turbeville.dna">Click here to download.</A>
<LI> A 76-species protein sequence data set for Transcription Elongation
Factor Alpha (EFTU-alpha), alignment by the Protein Information Resource
of the National Biomedical Research Foundation.
<A HREF="data/eftualpha.prt">Click here to download.</A>
Some species are represented more than once, and some by fragments.
<li> Here are 27 data sets, each the aligned DNA sequences of a different locus
for the same 40 mammalian species.  They were prepared for this class exercise
from the OrthoMAM database (they are almost all the loci on the 8th human
chromosome that were available for all these 40 species).  For each locus, the name is
the HUGO (Human
Genome Organisation) identifier (such as ADAM7).  You can get
information on that gene by a web search using "HUGO" and the identifier
(such as ADAM7) and the phrase "Gene Symbol Report".  A file
<a href="data/species">species</a> indicates the identities of the 40 species.
The files are in PHYLIP "Sequential" format.<br>
<table border=0 align=center>
<tr>
<td width="10%"><a href="data/ADAM7.dna">ADAM7.dna</a></td>
<td width="10%"><a href="data/GGH.dna">GGH.dna</a></td>
<td width="10%"><a href="data/RAD54B.dna">RAD54B.dna</a></td>
</tr>
<tr>
<td width="10%"><a href="data/AP3M2.dna">AP3M2.dna</a></td>
<td width="10%"><a href="data/GINS4.dna">GINS4.dna</a></td>
<td width="10%"><a href="data/RRM2B.dna">RRM2B.dna</a></td>
</tr>
<tr>
<td width="10%"><a href="data/ASH2L.dna">ASH2L.dna</a></td>
<td width="10%"><a href="data/GRHL2.dna">GRHL2.dna</a></td>
<td width="10%"><a href="data/SLC18A1.dna">SLC18A1.dna</a></td>
</tr>
<tr>
<td width="10%"><a href="data/CA8.dna">CA8.dna</a></td>
<td width="10%"><a href="data/GTF2E2.dna">GTF2E2.dna</a></td>
<td width="10%"><a href="data/SNX16.dna">SNX16.dna</a></td>
</tr>
<tr>
<td width="10%"><a href="data/CHMP4C.dna">CHMP4C.dna</a></td>
<td width="10%"><a href="data/KAT6A.dna">KAT6A.dna</a></td>
<td width="10%"><a href="data/STC1.dna">STC1.dna</a></td>
</tr>
<tr>
<td width="10%"><a href="data/DERL1.dna">DERL1.dna</a></td>
<td width="10%"><a href="data/LRP12.dna">LRP12.dna</a></td>
<td width="10%"><a href="data/TMEM55A.dna">TMEM55A.dna</a></td>
</tr>
<tr>
<td width="10%"><a href="data/E2F5.dna">E2F5.dna</a></td>
<td width="10%"><a href="data/MMP16.dna">MMP16.dna</a></td>
<td width="10%"><a href="data/TNKS.dna">TNKS.dna</a></td>
</tr>
<tr>
<td width="10%"><a href="data/ERLIN2.dna">ERLIN2.dna</a></td>
<td width="10%"><a href="data/NSMAF.dna">NSMAF.dna</a></td>
<td width="10%"><a href="data/WRN.dna">WRN.dna</a></td>
</tr>
<tr>
<td width="10%"><a href="data/FZD6.dna">FZD6.dna</a></td>
<td width="10%"><a href="data/OPRK1.dna">OPRK1.dna</a></td>
<td width="10%"><a href="data/WWP1.dna">WWP1.dna</a></td>
</tr>
</table>
<li> ... and here are the aligned protein sequences corresponding to them.  The
DNA and protein sequences are the "filtered" ones used by the OrthoMAM database
to infer their trees from the DNA sequences.  Some questionable regions have
been removed by them.<br>
<table border=0 align=center>
<tr>
<td width="10%"><a href="data/ADAM7.prt">ADAM7.prt</a></td>
<td width="10%"><a href="data/GGH.prt">GGH.prt</a></td>
<td width="10%"><a href="data/RAD54B.prt">RAD54B.prt</a></td>
</tr>
<tr>
<td width="10%"><a href="data/AP3M2.prt">AP3M2.prt</a></td>
<td width="10%"><a href="data/GINS4.prt">GINS4.prt</a></td>
<td width="10%"><a href="data/RRM2B.prt">RRM2B.prt</a></td>
</tr>
<tr>
<td width="10%"><a href="data/ASH2L.prt">ASH2L.prt</a></td>
<td width="10%"><a href="data/GRHL2.prt">GRHL2.prt</a></td>
<td width="10%"><a href="data/SLC18A1.prt">SLC18A1.prt</a></td>
</tr>
<tr>
<td width="10%"><a href="data/CA8.prt">CA8.prt</a></td>
<td width="10%"><a href="data/GTF2E2.prt">GTF2E2.prt</a></td>
<td width="10%"><a href="data/SNX16.prt">SNX16.prt</a></td>
</tr>
<tr>
<td width="10%"><a href="data/CHMP4C.prt">CHMP4C.prt</a></td>
<td width="10%"><a href="data/KAT6A.prt">KAT6A.prt</a></td>
<td width="10%"><a href="data/STC1.prt">STC1.prt</a></td>
</tr>
<tr>
<td width="10%"><a href="data/DERL1.prt">DERL1.prt</a></td>
<td width="10%"><a href="data/LRP12.prt">LRP12.prt</a></td>
<td width="10%"><a href="data/TMEM55A.prt">TMEM55A.prt</a></td>
</tr>
<tr>
<td width="10%"><a href="data/E2F5.prt">E2F5.prt</a></td>
<td width="10%"><a href="data/MMP16.prt">MMP16.prt</a></td>
<td width="10%"><a href="data/TNKS.prt">TNKS.prt</a></td>
</tr>
<tr>
<td width="10%"><a href="data/ERLIN2.prt">ERLIN2.prt</a></td>
<td width="10%"><a href="data/NSMAF.prt">NSMAF.prt</a></td>
<td width="10%"><a href="data/WRN.prt">WRN.prt</a></td>
</tr>
<tr>
<td width="10%"><a href="data/FZD6.prt">FZD6.prt</a></td>
<td width="10%"><a href="data/OPRK1.prt">OPRK1.prt</a></td>
<td width="10%"><a href="data/WWP1.prt">WWP1.prt</a></td>
</tr>
</table>

</ul>
<P>
Here are some other sources of aligned sequence data:
<ul>
<LI> A database of aligned mammalian gene sequences, OrthoMAM, is available
<a href="http://www.orthomam.univ-montp2.fr/orthomam/html/index.php?article=Request">here</a>.
Just click on the Submit button.  To see which protein each locus is, click on
the identifier field.
You can retrieve amino acid alignments in FASTA format.  You can copy and then paste the
sequences into <a href="http://www-bimas.cit.nih.gov/molbio/readseq/">this sequence
conversion web page</a> and choose "PHYLIP" format from its menu.
<LI> The <a href="http://www.pfam.org">Pfam</a> database of aligned
protein sequences is hosted by the European Bioinformatics institute.
Another way to use it is to use the <a
href="http://www.ebi.ac.uk/goldman-srv/pandit/">Pandit</a> web site which
makes Pfam alignments available in a number of formats.  One is the FASTA
format mentioned above.
</UL>
I will be putting up some more datasets and web sites of aligned
sequences.
<hr>
<a name="othercourses"> <h2> What are some other related courses?</h2>
</a><p>
<dt> Biology 354 <em>(Foundations in Evolution and Systematics)</em>
</dt><dd> The main evolution course at the University, taught twice yearly.
Last quarter (Autumn 2015) the course was given by
Jon Herron, co-author of the Freeman
and Herron evolution textbook.
In Winter 2015 this is given by Robert Bryson, a postdoctoral fellow who
works on phylogeography and biogeography.<br>
<DT> Genome 453 <em>(Evolutionary Genetics)</em>
<DD> This is an
undergraduate-level course on evolution and genetics.  Given every
Autumn, it overlaps somewhat with Biology 354 but
covers the genetic aspects of evolution with less emphasis on
ecology and paleontology than the first, less emphasis on molecular
evolution than the second.  In even-numbered years the course is given by
Mary Kuhner, a highly knowledeable population geneticist.
<i>It should be understood that this is basically an undergraduate course.</i>
<dt> Biology 414 <em> Molecular Evolution</em>
<dd> Theoretical and empirical study of molecular evolution.  Taught Winter,
2016 by Vladimir (Volodymyr) Minin of the Department of Biology and the
Department
of Statistics, who is actively involved in developing methods for statistical
and computationally analyzing molecular evolution. One 2-hour
lab session of Biology 414 overlaps with Genome 570 by 20 minutes.</dd>
</dd>
<dt> Biology 415 <em>(Evolution and development)</em></dt>
<dd> Advanced undergraduate 
course, requiring an introductory evolution course as a prerequisite, on
"Evo-devo", the intersection of developmental biology and evolution.  This has
been an area where there have been a lot of recent advances.
It was given in 2014; I am not sure when it will next be taught.
<dt> Biology 449 and Biology 553 <em>(Applied Phylogenetics)</em>
<dd> A course in applied aspects of phylogenetics,
with emphasis on exercises using actual data and widely-distributed programs.
Taught by Adam Leach&eacute;, who works on coalescents and phylogenies in
lizard species.  To be offered in Winter, 2016.
This course can be viewed as complementary to Genome 570.</dd>
<dt> Biology 469 <em>(Evolution and medicine)</em></dt>
<dd> A new course on
ways in which evolution provides insights and is applied in medicine.  Taught this quarter
by Daniel Promislow, an experienced evolutionary geneticist working in the
Department of Pathology in the School of Medicine.</dd>
<dt> Biology 481 <em>(Experimental Evolutionary Ecology)</em></dt>
<dd> Lecture and
lab course on evolutionary perspectives on ecology.  Taught by
Jake Cooper, who works in this area.  It will be given this quarter.</dd>
</dd><dt>Genome 562 <em>(Population Genetics)</em>
</dt><dd>This is the graduate theoretical
evolutionary genetics course that I give.  Lots of equations, though
mostly at a low mathematical level.  No pictures of cute furry animals.
The next time it will be given is in Winter, 2017, which may be the
last time it is given.  Text: my own extensive lecture
notes, downloaded as PDF's for free or sold inexpensively (no royalty is paid
to me).  </dd>
<dt> Biology 550A <i>(Evolution and Systematics Seminar)</i>
<dd> A seminar Fridays at 1:30pm covering various topics in evolution.  Runs
Autumn, Winter, and Spring quarters.</dd>
<dt> Biology 550B <i>(Paleo Lunch)</i> <dd>(No, the lunch is not fossilized).
A
lunchtime (Thursdays at 12:30) seminar on paleobiology. Autumn, Winter,
and Spring quarters.</dd>
<dt> Biology 550C <i>(Phylogenetics - Theory, Methods and Empirical Data Seminar)</i>
<dd> A seminar run by Adam Leach&eacute; Thursday at 2:30pm covering various topics in evolution.  Runs
Autumn, Winter, and Spring quarters.</dd>
</dd><dt> Others
</dt><dd> There are more courses and I'll gradually try to put descriptions of
them here.
</a></dd></dl>
<hr>
<a name="internet">
<h2> What are some Internet resources on evolutionary biology?</h2>
</a><p>
There are many:
<P>
<strong>Electronic journals</strong>
</a></p><p>
There is of course, the professional literature in evolutionary
biology. Contrary to popular belief, scientists don't publish their works
primarily by writing books.  They publish papers in scientific journals.
Some of these journals (links given below) are available in
electronic versions for UW people.  <b>If these links don't give you access
you should use the Electronic Journals links in the University Library site,
and type in the name of the journal</b>.  Here are some direct links to the
leading journals covering evolution:
</a></p><ul>
<li><a name="internet"> </a><a href="http://mbe.oupjournals.org/">Molecular Biology and Evolution</a>
<li> Systematic Biology
   <ul>
     <li> <a href="http://sysbio.oxfordjournals.org/">1998-present</a>
     <li> <a href="http://www.jstor.org/journals/10635157.html">1992-2007</a>
     <li> <a href="http://www.jstor.org/journals/00397989.html">1952-1992</a>
(as <em>Systematic Zoology</em>)
   </ul>
<li> Evolution
   <ul>
   <li> <a href="http://www.wiley.com/bw/journal.asp?ref=0014-3820&site=1">2000-present</a>
   <li> <a href="http://www.jstor.org/journals/00143820.html">1947-2007</a>
   </ul>
<li> <a href="http://link.springer-ny.com/link/service/journals/00239/tocs.html">Journal of Molecular Evolution (1997-present)</a>
</li><li> <a href="http://www.sciencedirect.com/science?_ob=JournalURL&_cdi=6963&_auth=y&_acct=C000029718&_version=1&_urlVersion=0">Molecular Phylogenetics and Evolution (1992-present) </a>
<li> Paleobiology
   <ul>
   <li> <a href="http://www.bioone.org/bioone/?request=get-journals-list&issn=0094-8373"> 1999-present </a>
   </li><li> <a href="http://www.jstor.org/journals/00948373.html"> 1975-2009 </a>
   </li></ul>
<li> <a href="http://www.blackwell-synergy.com/member/institutions/issuelist.asp?journal=jeb">Journal of Evolutionary Biology </a>
<li> <a href="http://www.jstor.org/journals/00030147.html">American Naturalist
</a> 1867-present
<li> <a href="http://www.biomedcentral.com/bmcevolbiol/">BMC Evolutionary
Biology</a> (Open Access) 2001-present
</DL>
<P>
There are also relevant papers on population genetics in genetics and genomics
journals such as Genetics, Genome Biology, Genome Research, Nature Genetics,
PLoS Genetics, BMC Genomics and Theoretical Population Biology.
</ul>
<P>
<STRONG> Blogs </STRONG>
<P>
There are blogs, mostly for creation / evolution debating.  There
also used to be Usenet newsgroups such as <tt>sci.bio.evolution</tt>.  The
latter are accessible through Google Groups, but they are mostly defunct now.
<P>
Some brief descriptions of some of the major ones covering evolution:
<DL>
<DT> <a href="http://www.pandasthumb.org">Panda's Thumb</a> (Blog)
<DD> Blog defending evolutionary biology against creationists and advocates of
Intelligent Design.
<DT> <a href="http://www.uncommondescent.com">Uncommon Descent</a> (Blog)
<DD> Blog centered around William Dembski and Denyse O'Leary, critics of
evolutionary biology and advocates of Intelligent Design.
<DT> <a href="http://www.evolutionnews.org">Evolution News and Views</a>
<DD>Does the name lead you to think that this is a blog by evolutionary biologists or science reporters?  Wrong.
It is a blog sponsored by the Discovery Institute, which promotes Intelligent
Design.  Don't get me started ...
</DL>
<P>
<strong>Web Pages</strong>
</p>
<ul>
<li> The amazing <a href="http://tolweb.org/tree"> Tree of Life </a>, a phylogeny of all life, in the
making.  Trees, figures, descriptions and references at a solid professional
level, each page done by a major systematist working on that group.  Much
more complete for some groups than others.  Explore!
<li> <a href="http://www.timetree.org">The TimeTree of Life</a> finds
estimated divergence times for many (pairs of) species.  Includes a lot
of downloadable PDF material including a whole book put out by this
NSF-sponsored project.
<li> <a href="http://www.treebase.org/treebase-web/home.html">TreeBASE</a>, a database of evolutionary trees (phylogenies)
from the scientific literature.  Weighted towards flowering plants, and
badly suffers being only sporadically updated.
<li> My own <a href="http://evolution.gs.washington.edu/phylip.html">
PHYLIP </a> free package of computer programs for inferring phylogenies.
<li> But maybe you don't want to be limited to just my programs?  OK,
try this listing which I maintain of all known
<a href="http://evolution.gs.washington.edu/phylip/software.html">Phylogeny
Programs</a>.  As of September 2011 there are 369 programs and 53 web servers
listed.  Many of the programs are freely downloadable.
<li> The <a href="http://www.paleoportal.org/">Paleontology Portal</a> run by
professional societies, the US Geological Survey, and the University of
California Museum of Paleontology gives information on many groups and their
fossils.
<li>The <a href="http://www.ucmp.berkeley.edu/">
University of California Museum of Paleontology </a>
pages, which a number of evolution and phylogeny electronic exhibits.  This is
a sort of "virtual museum".
<li> Want some DNA or protein sequences?  How about getting them from the
international database?  Try the <a href="http://www.ncbi.nlm.nih.gov/Database/index.html">Web pages of the databases</a> at the
NCBI, the National Center for Biotechnology Information of the National
Library of Medicine, in Bethesda, Maryland.  Not for the faint-hearted.
<li> Also not for the faint-hearted.  The <a href="http://genome.ucsc.edu">UCSC
Genome Browser</a> is the central site for the assembled human genome and
comparative genomics for other mammalian species.  See the genomics data
not as some textbook illustration, but as the genomic &ldquo;pros&rdquo; see it.
<li> <a href="http://www.orthomam.univ-montp2.fr/orthomam/html/">OrthoMam</a> is a good collection of most gene families
in mammals, aligned, and even with trees computed for each.  You can get
<i>lots</i> of data here.  A good test of the reality of common descent is to
see whether different genes give noticeably similar evolutionary trees (they
won't be exactly the same for a number of reasons, including noisy data).  Do
they?
<li> <a href="http://www.nationalacademies.org/evolution/WebResources.html">Evolution
web resources from the National Academy of Sciences</a>
aimed at the general public or scientifically knowledgable laypersons.
<li> The <a href="http://www.talkorigins.org/origins/faqs.html">
talk.origins Evolution FAQs</a> (Frequently Asked Questions).  These are
FAQ pages started for the former talk.origins newsgroup, where evolution was
defended against creationism.  They have been expanded since and are generally
a very complete resource for evaluating all sorts of creationist arguments.
<li>Kent Holsinger's <A HREF="http://darwin.eeb.uconn.edu/simulations/simulations.html">
Population Biology Simulations</A>, some Java applets to illustrate various
phenomena.
<li> The <a href="http://mcb.harvard.edu/BioLinks/Evolution.html">Harvard Biopages evolution page</a>, with lots of links to other
evolution resources.  But it has not been maintained and many links are broken.
</UL>
<P>
<hr>
This page maintained fitfully by Joe Felsenstein
</body></html>
