<HEAD>
<meta http-equiv="Content-Type" content="text/html; charset=ISO-8859-1">
<TITLE>Typographical errors in "Inferring Phylogenies"</TITLE>
<LINK href="../bluestyle.css" rel="stylesheet" type="text/css">
</HEAD>

<BODY BGCOLOR="#ffee99">
<DIV ALIGN=CENTER>
<H1>Typos in "Inferring Phylogenies"</H1>
<BR>
<P>
<DIV ALIGN=LEFT>
Six printings have now appeared.  They are in two groups.  Before Sinauer
Associates was acquired by Oxford University Press, the name "Oxford University
Press" does not appear on the back cover of the book.  For those
printings, to see which printing you have,
look on the copyright page (the page after the title page).  At the
bottom is a row of digits (5 4 3 2 etc.).  The last one is the number of
the printing.  The third printing also has a light blue background in the
cover photo of the Far Flung Fir sculpture (all other
editions have an olive-colored background).  The first printing can also be 
distinguished by the absence of quotes from reviewers on the back cover, which 
is mostly just black.   There never was a fifth printing -- it was
accidentally skipped, as the printing after the 4th printing was numbered 6.
<P>
The printings that say "Oxford University Press" on the back cover are the
next ones, and the numbers of the printings will be 2, 3, ...  So I'll call
the initial Oxford University Press printing the "second Oxford University
Press printing".
</UL>
</DIV>
<P>
Contents of this web page:
<a href="#secondOUP"><h3>Typos in second Oxford University Press printings and all
previous printings</h3></a>
<a href="#sixth"><h3>Typos in the first, second, third, fourth, and sixth printings</h3></a>
<a href="#second"><h3>Typos in the first and second printings</h3></a>
<a href="#first"><h3>Typos in the first printing</h3></a>
<a href="#references"><h3>References that should have been in the book</h3></a>
<a href="#erroneous"><h3>Erroneous statements in the book</h3></a>
<a href="#2ndedition"><h3>A second edition?</h3></a>
<BR>
<BR>
<P>
The following typographical or algebraic errors are known.  They are given by page.
<P>
<a name="secondOUP"><h3>Typos in second Oxford University Press printing and all previous printings</h3></a>
<div align=center>
<P>
These are in all the printings up to and including the second OUP printing.  The exception is the first correction
for page 396, which is a new problem in the second OUP printing.
</div>
<! number of typos = 9>
<table border=1>
<tr bgcolor=silver><td colspan=3><font color=blue>Corrections in
book text, figures, or index</font></td></tr>
<tr> <td>105</td> <td> in other case as well </td> <td> in other cases as well</td> </tr>
<tr> <td>149</td> <td>(in matrix in figure)</td> <td>distance between C and D should be 0.20</td> </tr>
<tr> <td>179</td> <td>should not be the largest</td> <td>should be the smallest</td> </tr>
<tr> <td>242</td> <td>&mu;t = 10</td> <td>&mu;t = 5</td> </tr>
<tr> <td>310</td> <td>(in equation 19.7)</td> <td>the second&nbsp; &theta;<sub>0</sub>&nbsp; should be &nbsp;&theta;&nbsp  </td> </tr>
<tr> <td>396</td> <td>(add sentence to Figure caption)</td><td>The tree has the same topology as Figure 23.1.</td></tr>
<tr><td>396</td><td>(Figure caption abuts text).</td><td>(Need to shrink figure a bit so there is space).
<tr> <td>461</td> <td>(in equation 26.11)</td> <td>both occurrences of <em>N<sub>e</sub></em> should be <em>N<sub>0</sub></em></td> </tr>
<tr><td>645</td><td>Index</td><td>References</td></tr>
</table>
<P>
<a name="sixth"><H3>Typos in the first, second, third, fourth, and sixth printings</H3></a>
<p>
<div align=center>
These are in all the printings up to and including the sixth printing. 
They are corrected in the second OUP printing.
</div>
<p>
<! number of typos = 78>
<table border=1>
<tr bgcolor=silver><td colspan=3><font color=blue>Corrections in
book text, figures, or index</font></td></tr>
<tr bgcolor=silver> <td> <font color=blue> page </font> </td> <td> <font color=blue> phrase (or location): </font> </td> <td> <font color=blue> should be: </font> </td> </tr>
<tr> <td>&nbsp;&nbsp;20</td> <td>We will consider a building up</td> <td>We will consider building up</td> </tr>
<tr> <td>&nbsp;&nbsp;22</td> <td>(in Figure 3.3)</td> <td>The branch of the top
two-species tree where<br> species <em>c</em> is added to
get the center<br> three-species tree is the wrong one.</td> </tr>
<tr><td>&nbsp;&nbsp;66</td><td>there are not or more two states that both occur<br>
<i>(this one was different in the first edition)</i></td><td>there are
not two or more states that occur</td></tr>
<tr> <td>107</td> <td>homplasious</td> <td>homoplasious</td> </tr>
<tr> <td>113</td> <td>that the probability becomes 1</td> <td>the probability
becomes 1</td> </tr>
<tr> <td>119</td> <td>in both bases the curves</td> <td>in both cases the curves</td></tr>
<tr> <td>129</td> <td>with affinities to schmoos</td> <td>with affinities to shmoos</td></tr>
<tr> <td>132</td> <td><i>(in figure caption)</i> cytochrome</td> <td>globin</td></tr>
<tr> <td>152</td> <td>Gascuel (1997)</td> <td>Gascuel (1997b)</td> </tr>
<tr> <td>157</td> <td>it will certainly rise linearly</td> <td>it will certainly not rise linearly</td> </tr>
<tr> <td>170</td> <td>Gascuel (1997)</td> <td>Gascuel (1997a)</td></tr>
<tr> <td>174</td> <td>Gascuel, Desper, and Denis</td> <td>Gascuel, Bryant, and
Denis</td></tr>
<tr> <td>178</td> <td>more then four</td> <td>more than four</td></tr>
<tr> <td>178</td> <td>D<sub>il</sub>+D<sub>ij</sub></td>
<td>D<sub>il</sub>+D<sub>jk</sub> </td> </tr>
<tr> <td>179</td> <td>(in equation 12.7) &nbsp;&nbsp;&nbsp;max</td> <td>min</td> </tr>
<tr> <td>188</td> <td>(in Table 12.2) </td> <td>The
numbers 10 in the bottom three rows should be 5.</td> </tr>
<tr> <td>226</td> <td>only between amino acids can be reached</td> <td>only
between amino acids that can be reached </td> </tr>
<tr> <td>236</td> <td>more defensible that it proved</td> <td>more
defensible than it proved</td> </tr>
<tr> <td>241</td> <td>high rates of change has</td> <td>high rates of change have</td> </tr>
<tr> <td>250</td> <td>(in caption of Fig. 16.1) series of 13 independent&nbsp;&nbsp;&nbsp;&nbsp;</td> <td>series of 11 independent</td> </tr>
<tr> <td>269</td> <td>states will be spend</td> <td>states will spend</td> </tr>
<tr> <td>269</td> <td>as as it will with a</td> <td>as they will with a</td> </tr>
<tr> <td>269</td> <td>and site being variable</td> <td>and the site being variable</td> </tr>
<tr> <td>310</td> <td>(in equation 19.7)</td> <td><b>C</b> should be (<b>-C</b>)</td> </tr>
<tr> <td>310</td> <td>(in equation 19.8)</td> <td><b>C</b> should be (<b>-C</b>)</td> </tr>
<tr> <td>310</td> <td>that can takes values</td> <td>that can take values</td> </tr>
<tr> <td>313</td> <td>have been used give a good approximation</td> <td>have been used to give a good
approximation</td></tr>
<tr> <td>318</td> <td><i>(in the Figure)</i></td> <td>quantities T should be R</td> </tr>
<tr> <td>327</td> <td>noted, in Chapter 16 that</td> <td>noted in Chapter 16 that</td> </tr>
<tr> <td>328</td> <td>for each these simulated data sets</td> <td>for each of these simulated data sets</td> </tr>
<tr> <td>337</td> <td>we would back get our original data</td> <td>we would get back our original data</td> </tr>
<tr> <td>340</td> <td>0.68066 of the time</td> <td>0.680905 of the time</td> </tr>
<tr> <td>344</td> <td> have randomly assigned rate</td> <td>each have a
randomly assigned rate</td> </tr>
<tr> <td>348</td> <td> (the one-tailed 95% point </td> <td>(the two-tailed 95%
point </td> </tr>
<tr> <td>349</td> <td> even greater when consider</td> <td>even greater when we consider</td> </tr>
<tr> <td>351</td> <td>that lead to</td> <td>that led to</td> </tr>
<tr> <td>352</td> <td>with more then R<sup>3</sup> replicates</td> <td>with more than R<sup>3</sup> replicates</td> </tr>
<tr> <td>355</td> <td> we find ABCEDF</td> <td>we find ABCDEF</td> </tr>
<tr> <td>371</td> <td>Data sets generated</td> <td>Data sets are generated</td> </tr>
<tr> <td>386</td> <td>seven degrees of freedom</td> <td>six degrees of freedom</td> </tr>
<tr> <td>386</td> <td>all seven</td> <td>all six</td> </tr>
<tr> <td>388</td> <td>Then number of equations</td> <td>The number of equations</td> </tr>
<tr> <td>395</td> <td>tree has the same topology as Figure 23.2</td> <td>tree has the same topology as Figure 23.1</td> </tr>
<tr> <td>396</td> <td>The tree has the same topology as Figure 23.1.</td>
<td><i>(delete the sentence)</i></td> </tr>
<tr> <td>402</td> <td><i>i</i>th character in the <i>j</i>th population</td>
<td><i>j</i>th character in the <i>i</i>th population</td> </tr>
<tr> <td>402-403</td>
<td>The values of all <i>n</i> characters<br>
in population 1 are on top.<br>
Below them are the values for all <i>n</i><br>
characters in population 2, and so on.</td>
<td>The values for all <i>n</i> populations<br>
of character 1 are on top.<br>
Below them are the values for all <i>n</i><br>
populations of character 2, and so on.</td>
</tr>
<tr> <td>403</td> <td>an <i>p &times p</i> array</sup></td> <td>a <i>p &times;
p</i> array</td> </tr>
<tr> <td>403</td> <td><i>(in Figure 23.34)</i> V<sup>(n)</sup></td> <td>V<sup>(p)</sup></td> </tr>
<tr> <td>403</td> <td><i>(in equation 23.36)</i> (mean vector 0 should be)</td>
<td> (a vector &mu; Kronecker-product with the identity matrix I)</td> </tr>
<tr> <td>403</td> <td>an <i>(n-1)p &times; (n-1)p</i> matrix</td> <td>an
<i>(n-1)p &times; np</i> matrix</td> </tr>
<tr> <td>410</td> <td>Gegenbaur</td> <td>Gegenbauer</td></tr>
<tr> <td>418</td> <td>2N times the variance of change</td> <td>N times the
variance of change</td> </tr>
<tr> <td>425</td> <td>It seem that there is</td> <td>It seems that there is</td> </tr>
<tr> <td>430</td> <td>reversion to state 1</td> <td>reversion to state 0</td> </tr>
<tr> <td>435</td> <td>1.11666</td> <td>1.1666</td> </tr>
<tr> <td>436</td> <td>1.11666</td> <td>1.1666</td> </tr>
<tr> <td>471</td> <td>two occurrences of <i>k<sub>i</sub></i></td> <td>should
both be <i>k<sub>li</sub></i></td> </tr>
<tr> <td>471</td> <td>two occurrences of <i>m<sub>jl</sub></i></td> <td>should
both be <i>m<sub>lj</sub></i></td> </tr>
<tr> <td>472</td> <td><i>n-1, n-2, ..., 2</i> lineages</td> <td><i>n, n-1, ...,
2</i> lineages</td></tr>
<tr> <td>512</td> <td>in its immediate ancestors</td> <td>in its immediate
descendants</td> </tr>
<tr> <td>512</td> <td>shows three unrooted trees</td> <td>shows two rooted
trees</td> </tr>
<tr> <td>514</td> <td>all possible number of loci</td> <td>all possible numbers of loci</td> </tr>
<tr> <td>521</td> <td>placement of their root. the two trees</td> <td>placement of their roots, the two trees</td> </tr>
<tr> <td>559</td> <td>test hypothesis about evolution</td> <td>test hypotheses about evolution</tr>
<tr> <td>563</td> <td>tips in the tree, or N-bar its mean</td> <td>tips in the tree, or N-bar, its mean</td> </tr>
<tr> <td>563</td> <td>compared it to a the probabilities</td> <td>compared it to the probabilities</td> </tr>
<tr> <td>568</td> <td>(in equation 33.6) &nbsp;&nbsp;&nbsp;..., t<sub>n</sub></td> <td>&nbsp;&nbsp;&nbsp;..., t<sub>n-1</sub></td> </tr>
<tr> <td>576</td> <td>lowermost ancestor is used</td> <td>lowermost descendant
is used</td> </tr>
<tr> <td>588</td> <td>one calls function <tt>traverse</tt> or
<tt>Traverse</tt></td> <td> one calls function <tt>postorder</tt> or
<tt>Traverse</tt></td> </tr>
<tr> <td>604</td>
<td> (Edwards 1971 reference)  The likelihood for<br> multinomial proportions
under stereographic<br>
projection. <i>Biometrics</i> <b>28:</b> 618-620</td>
 <td>Distance between
populations on
the basis of<br> gene frequencies.  <i>Biometrics</i> <b>27:</b> 873-881. </td> </tr>
<tr> <td>611</td> <td> (the two listings for Gascuel, 1997)</td> <td> the first should be
1997a, the second 1997b</td> </tr>
<tr> <td>614</td> <td>(Reference to Hein, J. 1989), title should be:
<td>A new method that simultaneously aligns and<br>
reconstructs ancestral sequences for any number of<br>
homologous sequences, when the phylogeny is given.</td> </tr>
<tr> <td>616</td> <td>(Reference to Hudson, R. 1983 in <em>Evolution</em>)</td>
<td>This reference needs to start on its own<br>
   (negatively indented) line.</td> </tr>
<tr> <td>627</td> <td>(Reference to Nielsen, R. 2002)</td>
<td>volume number needs to be in boldface type</td> </tr>
<tr> <td>633</td> <td>(Reference to Sankoff, Morel,<br> and Cedergren, 1973)</td>
<td>volume number needs to be in boldface type</td> </tr>
<tr> <td>638</td> <td>Swofford, D. L. and D. R. Maddison</td> <td>Swofford, D.
L. and W. P. Maddison</td> </tr>
<tr> <td>639</td> <td>(Thompson 1972 reference) Distance<br> between
populations on
the basis of<br> gene frequencies.  <i>Biometrics</i> <b>27:</b> 873-881. </td>
<td>The likelihood for multinomial proportions<br> under stereographic
projection.<br> <i>Biometrics</i> <b>28:</b> 618-620</td> </tr>
<tr> <td>658</td> <td>indexing of Ronquist, 2000</td> <td>(should have been on
page 659<br> among the other Ronquist page numbers)</td> </tr>
</table>
<P>
<a name="second"><H3>Typos in the first and second printings</H3></a>
<P>
<div align=center>
The following typographical errors are found in the first and second printings.
They are corrected in the third printing.
</div>
<P>
<table border=1>
<tr bgcolor=silver><td colspan=3><font color=blue>Corrections in
book text, figures, or index</font></td></tr>
<tr bgcolor=silver> <td> <font color=blue> page </font> </td> <td> <font color=blue> phrase (or location): </font> </td> <td> <font color=blue> should be: </font> </td> </tr>
<! number of typos =  45>
<tr> <td>&nbsp;&nbsp;2</td> <td>Figure 1.4 shows the single reconstruction</td> <td>Figure 1.4 shows the two reconstructions</td> </tr>
<tr> <td>&nbsp;&nbsp;2</td> <td>(in Figure 1.1)</td> <td>Raise label Delta to be level with the others</td> </tr>
<tr> <td>&nbsp;33</td> <td>In Table 3.5, many of the lines for even<br>
numbers of species were computed incorrectly</td> <td>The corrected lines are (showing<br>
only the ones that have changed):<br>
<table><tr><td align=right>8</td><td align=right>32</td><td align=right>4</td></tr>
<tr><td>10</td><td align=right>190</td><td align=right>11</td></tr>
<tr><td>12</td><td align=right>1,350</td><td align=right>37</td></tr>
<tr><td>14</td><td align=right>10,765</td><td align=right>135</td></tr>
<tr><td>16</td><td align=right>92,949</td><td align=right>552</td></tr>
<tr><td>18</td><td align=right>847,511</td><td align=right>2,410</td></tr>
<tr><td>20</td><td align=right>8,044,399</td><td align=right>11,020</td></tr>
<tr><td>30</td><td align=right>8.913 x 10<sup>11</sup></td><td align=right>3.294 x 10<sup>7</sup></td></tr>
<tr><td>40</td><td align=right>&nbsp;&nbsp;&nbsp;&nbsp;1.377 x 10<sup>17</sup></td><td align=right>&nbsp;&nbsp;&nbsp;&nbsp;1.385 x 10<sup>11</sup></td></tr>
</table></td></tr>
<tr> <td>&nbsp;35</td> <td>now [(n-1)-1][(n-2)-1]/2 =</td> <td>now [n-1][(n-1)-1]/2 =</td> </tr>
<tr> <td>102</td> <td>they arive at</td> <td>they arrive at</td> </tr>
<tr> <td>113</td> <td>as long as <em>q < 1/2</em>, <em>p > 0</em>, and <em>q > 0</em></td> <td>as long as <em>q < 1/2</em>, and either <em>p > 0</em> or <em>q > 0</em></td> </tr>
<tr> <td>123</td> <td>carried our by a computer</td> <td>carried out by a computer</td> </tr>
<tr> <td>134</td> <td>prefer not to assume either reversibility</td> <td>prefer not to assume either irreversibility</td> </tr>
<tr> <td>155</td> <td>(in equation 11.14)&nbsp;&nbsp;&nbsp;<b>D</b></td> <td>&nbsp;&nbsp;<b>d</b></td> </tr>
<tr> <td>155</td> <td>(in equation 11.15)&nbsp;&nbsp;&nbsp;<b>D</b></td> <td>&nbsp;&nbsp;<b>d</b></td> </tr>
<tr> <td>156</td> <td>of length <em>t</em></td> <td>of time <em>t</em></td> </tr>
<tr> <td>156</td> <td>of length <em>dt</em> each</td> <td>of time <em>dt</em> each</td> </tr>
<tr> <td>161</td> <td>is then the average of the distances </td> <td>is then half the average of the distances</td> </tr>
<tr> <td>161</td> <td>is the average of the distances</td> <td>is half the average of the distances</td> </tr>
<tr> <td>172</td> <td>the neighbor-Joining methods</td> <td>the neighbor-joining methods</td> </tr>
<tr> <td>177</td> <td>see the papers by Gascuel</td> <td>see the paper by Gascuel</td> </tr>
<tr> <td>200</td> <td>(in equation 13.4)&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<em>Q<sup><sup>n<sub>2</sub></sup></sup></em></td><td><font size=+2>(</font><sup><em>1</sup>/<sub>2</sub>&nbsp;Q<font size=+2>)</font><em><sup>n<sub>2</sub></sup></sup></em></td> </tr>
<tr> <td>203</td> <td> (In numerator of second line of equation 13.11) </td> <td> the epsilon should be the same style<br> as the one in the denominator </td> </tr>
<tr> <td>213</td> <td>all the bases frequencies</td> <td>all the base frequencies</td> </tr>
<tr> <td>221</td> <td>Rzhetsky, 1995, Tillier, and Collins, 1998</td> <td>Rzhetsky, 1995; Tillier and Collins, 1998</td> </tr>
<tr> <td>229</td> <td>Pagel (1995)</td> <td>Pagel (1994)</td> </tr>
<tr> <td>245</td> <td>(in equation 15.28) &nbsp;&nbsp;max</td> <td>min</td> </tr>
<tr> <td>261</td> <td>amounts of diveregence</td> <td>amounts of divergence</td> </tr>
<tr> <td>279</td> <td>each entry in the this vector</td> <td>each entry in this vector</td> </tr>
<tr> <td>294</td> <td colspan=2>Figure 18.2 should have its upper-right circle open to
the path that comes in from its lower left.</td> </tr>
<tr> <td>332</td> <td>the the confidence limits</td> <td>the confidence limits</td> </tr>
<tr> <td>355</td> <td>if we could Type I</td> <td>if we count Type I</td> </tr>
<tr> <td>382</td> <td>the triple of species <em>a</em>, <em>b</em>, and <em>c</em></td> <td>the triple of species <em>c</em>, <em>d</em>, and <em>e</em></td> </tr>
<tr> <td>410</td> <td>the Ornstein-Uhlenbeck process<br>(which will be described later in this chapter)</td> <td>the Ornstein-Uhlenbeck process<br>(which will be described in the next chapter)</td> </tr>
<tr> <td>418</td> <td>(in equation 24.6)&nbsp;&nbsp;&nbsp;Var[g-bar]</td> <td>&nbsp;&nbsp;Var[&Delta; g-bar]</td> </tr>
<tr> <td>430</td> <td colspan=2>(in equation 24.18)&nbsp; The last integral should be from -infinity to 0.</td> </tr>
<tr> <td>438</td> <td><em>x<sub>1</sub>-x<sub>2</sub></em> multiplied by</td> <td><em>x'<sub>1</sub>-x'<sub>2</sub></em> multiplied by</td> </tr>
<tr> <td>491</td> <td>they give is no way</td> <td>they give no way</td> </tr>
<tr> <td>500</td> <td>Wang and Jiang (1994</td> <td>Wang and Jiang (1993)</td> </tr>
<tr> <td>532</td> <td>(in Figure 30.8)&nbsp;&nbsp;&nbsp;{BC&nbsp;|&nbsp;ADEFG}&nbsp;&nbsp;&nbsp;&nbsp;0.2&nbsp;&nbsp;&nbsp;&nbsp;0.2</td> <td>&nbsp;&nbsp;&nbsp;{BC&nbsp;|&nbsp;ADEFG}&nbsp;&nbsp;&nbsp;&nbsp;0.2&nbsp;&nbsp;&nbsp;&nbsp;0.3</td> </tr>
<tr> <td>532</td> <td>(in equation 30.1)<br>&nbsp;|&nbsp;0 - 0.2&nbsp;|&nbsp; + &nbsp;|&nbsp;0.4 - 0.3&nbsp;|&nbsp;  + &nbsp; |&nbsp;0.3 - 0&nbsp;|&nbsp; + &nbsp;|&nbsp;0.1 - 0&nbsp;|&nbsp;<br>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; + &nbsp;|&nbsp;0.05 - 0.2&nbsp;|&nbsp; + &nbsp;|&nbsp;0.2 - 0.1&nbsp;|&nbsp;=&nbsp;0.95</td>  <td>&nbsp;&nbsp;&nbsp;|&nbsp;0  -  0.2&nbsp;|&nbsp; + &nbsp;|&nbsp;0.4 - 0.3&nbsp;|&nbsp; + &nbsp;|&nbsp;0.2 - 0.3&nbsp;|&nbsp;  +  &nbsp;|&nbsp;0.3 - 0&nbsp;|<br>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; + &nbsp;|&nbsp;0.1 - 0&nbsp;|&nbsp; + &nbsp;|&nbsp;0.05 - 0.2&nbsp;|&nbsp; + &nbsp;|&nbsp;0.2 - 0.1&nbsp;|&nbsp;=&nbsp;1.05</td> <td></td> </tr>
<tr> <td>532</td> <td>(in equation 30.2)<br>&nbsp;(0 - 0.2)<sup>2</sup>&nbsp; + &nbsp;(0.4 - 0.3)<sup>2</sup>&nbsp;  + &nbsp; (0.3 - 0)<sup>2</sup>&nbsp; + &nbsp;(0.1 - 0)<sup>2</sup>&nbsp;<br>&nbsp;&nbsp;&nbsp; + &nbsp;(0.05 - 0.2)<sup>2</sup>&nbsp; + &nbsp;(0.2 - 0.1)<sup>2</sup>&nbsp;=&nbsp;0.1825</td>  <td>&nbsp;&nbsp;&nbsp;(0  -  0.2)<sup>2</sup>&nbsp; + &nbsp;(0.4 - 0.3)<sup>2</sup>&nbsp; + &nbsp;(0.2 - 0.3)<sup>2</sup>&nbsp;  +  &nbsp;(0.3 - 0)<sup>2</sup><br>&nbsp;&nbsp; + &nbsp;(0.1 - 0)<sup>2</sup>&nbsp; + &nbsp;(0.05 - 0.2)<sup>2</sup>&nbsp; + &nbsp;(0.2 - 0.1)<sup>2</sup>&nbsp;=&nbsp;0.1925</td> <td></td> </tr>
<tr> <td>563</td> <td>tree in Figure 33.1 this is 7/72</td> <td>tree in Figure 33.1 this is 14/72</td> </tr>
<tr> <td>578</td> <td>will attached be three or more subtrees</td> <td>will be three or more subtrees attached</td> </tr>
<tr> <td>581</td> <td>the branches leading to D and B</td> <td>the branches leading to D and E</td> </tr>
<tr> <td>586</td> <td>composed to records</td> <td>composed of records</td> </tr>
<tr> <td>587</td> <td>in turns of itself</td> <td>in terms of itself</td> </tr>
<tr> <td>591</td> <td>&lt;clade length=0.6&gt&nbsp;&nbsp;<br>&nbsp;&nbsp;&nbsp;&lt;clade length=0.102&gt;&lt;name&gt;A&lt;/name&gt;<br>&nbsp;&nbsp; &lt;clade length=0.23&gt;&lt;name&gt;B&lt;/name&gt;<br>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;and<br>&nbsp;&nbsp;&lt;clade length=0.4&gt;&lt;name&gt;C&lt;/name&gt;&lt;/clade&gt;</td> <td>&lt;clade length="0.6"&gt;<br>&nbsp;&nbsp; &lt;clade length="0.102"&gt;&lt;name&gt;A&lt;/name&gt;&lt;/clade&gt;<br>&nbsp;&nbsp; &lt;clade length="0.23"&gt;&lt;name&gt;B&lt;/name&gt;&lt;/clade&gt;<br>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;and<br>&nbsp;&nbsp;&lt;clade length="0.4"&gt;&lt;name&gt;C&lt;/name&gt;&lt;/clade&gt;
<tr> <td>593</td> <td>The URL given for TNT is outdated</td> <td>It should instead be:<BR>
&nbsp;&nbsp;<tt>http://www.zmuc.dk/public/phylogeny/tnt</tt></td> </tr>
<tr> <td>615</td> <td>(in Hendy and Penny 1982 reference) &nbsp;&nbsp;&nbsp;<b>60:</b> 133-142.</td> <td><b>59:</b> 277-290.</td> </tr>
 </td> </tr>
</table>
<P>
<a name="first"><H3>Typos in the first printing</H3></a>
<! current number of typos = 261>
<P>
<div align=center>
The first printing contains the above errors plus these:
</div>
<P>
<table border=1>
<tr bgcolor=silver><td colspan=3><font color=blue>Corrections of page numbers in Table of Contents entries</font></td></tr>
<tr bgcolor=silver> <td> <font color=blue> page </font> </td> <td> <font color=blue> in entry: </font> </td> <td> <font color=blue> page number should be: </font> </td> </tr>
<tr> <td>v</td><td>Tree shapes, p. 28</td><td>p. 29</td></tr>
<tr> <td>vi</td><td>Genetic algorithms, p. 44</td><td>p. 45</td></tr>
<tr> <td>vi</td><td>Tree space, p. 48</td><td>p. 50</td></tr>
<tr> <td>vi</td><td>NP-hardness, p. 57</td><td>p. 59</td></tr>
<tr> <td>vii</td><td>Nonsuccessive algorithms, p. 84 </td><td>p. 85 </td></tr>
<tr> <td>vii</td><td>Observed fractions of the patterns, p. 110</td><td>p. 111</td></tr>
<tr> <td>vii</td><td>Camin-Sokal and parsimony, p. 128</td><td>p. 129</td></tr>
<tr> <td>viii</td><td>Hypothetico-deductive, p. 138</td><td>p. 139</td></tr>
<tr> <td>viii</td><td>Logical parsimony?, p. 140</td><td>p. 141</td></tr>
<tr> <td>viii</td><td>The Jukes-Cantor model - an example, p. 155</td><td>p. 156</td></tr>
<tr> <td>ix</td><td>Codon-based models, p. 225</td><td>p. 226</td></tr>
<tr> <td>xi</td><td>Bayesian methods for phylogenies, p. 289</td><td>p. 291</td></tr>
<tr> <td>xi</td><td>Flat priors and doubts about them, p. 301</td><td>p. 302</td></tr>
<tr> <td>xi</td><td>Testing assertions about parameters, p. 311</td><td>p. 312</td></tr>
<tr> <td>xii</td><td>Parsimony-based methods, p. 322</td><td>p. 323</td></tr>
<tr> <td>xii</td><td>Tests for three species with a clock, p. 329</td><td>p. 330</td></tr>
<tr> <td>xii</td><td>Independence of characters, p. 342</td><td>p. 343</td></tr>
<tr> <td>xii</td><td>Permutation tests, p. 358</td><td>p. 359</td></tr>
<tr> <td>xii</td><td>The SH test, p. 369</td><td>p. 370</td></tr>
<tr> <td>xiii</td><td>Symmetry invariants, p. 374</td><td>p. 375</td></tr>
<tr> <td>xiii</td><td>Finding all invariants empirically, p. 387</td><td>p. 388</td></tr>
<tr> <td>xiii</td><td>The REML approach, p. 400</td><td>p. 401</td></tr>
<tr> <td>xiii</td><td>Using approximate Brownian motion, p. 411</td><td>p. 412</td></tr>
<tr> <td>xiv</td><td>Selection for an optimum, p. 420</td><td>p. 421</td></tr>
<tr> <td>xv</td><td>Fu's method, p. 484</td><td>p. 485</td></tr>
<tr> <td>xv</td><td>Methods of inferring the species phylogeny, p. 490</td><td>p. 491</td></tr>
<tr> <td>xv</td><td>Parsimony method, p. 497</td><td>p. 498</td></tr>
<tr> <td>xv</td><td>Reconciled trees, p. 509</td><td>p. 510</td></tr>
<tr> <td>xvi</td><td>Strict consensus, p. 521</td><td>p. 522</td></tr>
<tr> <td>xvi</td><td>The symmetric difference, p. 528</td><td>p. 529</td></tr>
<tr> <td>xvi</td><td>Stratocladistics, p. 549</td><td>p. 550</td></tr>
<tr> <td>xvii</td><td>Index, p. 644</td><td>p. 645</td></tr>
</table>
<P>
<table border=1>
<tr bgcolor=silver><td colspan=3><font color=blue>Corrections in
book text, figures, or index</font></td></tr>
<tr bgcolor=silver> <td> <font color=blue> page </font> </td> <td> <font color=blue> phrase (or location): </font> </td> <td> <font color=blue> should be: </font> </td> </tr>
<tr> <td> xiii </td> <td>23 Brownian motion and<br>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;gene frequencies </td> <td>23 Brownian motion and gene frequencies</td> </tr>
<tr> <td> 13 </td> <td>only A or G are possible </td> <td>only A or C are possible </td> </tr>
<tr> <td> 15 </td> <td>In Figure 2.2, the fourth tip<br>has: &nbsp;&nbsp; infinity, infinity, infinity, 0</td> <td>should be: &nbsp;&nbsp; 0, infinity, infinity, infinity </td> </tr>
<tr> <td> 15 </td> <td> these total to 2.5 </td> <td> these total to 4.5 </td> </tr>
<tr><td>16</td> <td>Wheeler and Nixon (1990)</td><td>Wheeler and Nixon (1994)</td></tr>
<tr> <td> 23 </td> <td> &nbsp;&nbsp;(2n-3)!<br> -------------<br> 2<sup>n-1</sup> (n-1)!  </td> <td> &nbsp;&nbsp;(2n-3)!<br> -------------<br> 2<sup>n-2</sup> (n-2)!  </td> </tr>
<tr><td>24</td><td>rise to a rooted bifurcating trees</td><td>rise to rooted bifurcating trees</td></tr>
<tr> <td>29</td> <td>Edwards and Cavalli-Sforza</td> <td>Cavalli-Sforza and
Edwards</td> </tr>
<tr> <td>30</td> <td>(in Table caption):&nbsp;&nbsp;&nbsp;&nbsp;Edwards and Cavalli-Sforza</td> <td>Cavalli-Sforza and
Edwards</td> </tr>
<tr> <td> 35 </td> <td> connected with of genes </td> <td> connected with coalescent trees of genes </td> </tr>
<tr> <td> 36 </td> <td> program and want to know </td> <td> program and wants to know </td> </tr>
<tr><td>38</td><td>because they sieze the first</td><td>because they seize the first</td></tr>
<tr> <td>42</td> <td>(in figure caption)&nbsp;&nbsp;&nbsp;separates B and C</td> <td>separates G and C</td> </tr>
<tr> <td> 44 </td> <td> 304 </td> <td> 288 </td> </tr>
<tr> <td> 62 </td> <td>(which requires 3 changes) </td> <td>(which requires 5 changes) </td> </tr>
<tr> <td> 63 </td> <td>(in Figure 5.4)&nbsp;&nbsp;&nbsp;bottommost node has number 3 </td> <td>should be 5 </td> </tr>
<tr><td>66</td><td>there are not two states that both occur</td><td>there are not two or more states that occur</td></tr>
<tr><td>69</td><td>reconstruction of states at interior nodes is</td><td>reconstructions of states at interior nodes are</td></tr>
<tr><td>73</td><td>any one of them are present</td><td>any one of them is present</td></tr>
<tr> <td> 75</td> <td> (in middle row of equation 7.1)&nbsp;&nbsp;&nbsp;<em>x<sub>l</sub>&nbsp;&le;&nbsp;x<sub>r</sub></em></td>
<td>&nbsp;&nbsp; <em>x<sub>l</sub>&nbsp;&le;&nbsp;x&nbsp;&le;&nbsp;x<sub>r</sub></em></td></tr>
<tr> <td> 79 </td> <td> (in figure caption) which shows that </td> <td> that shows which </td> </tr>
<tr> <td> 88 </td> <td>(in Table 8.2)</td> <td>the table should be transposed so that<br> the 01 combination is the one missing</td> </tr>
<tr> <td> 92 </td> <td> Table 1.1 </td> <td> Table 8.1 </td> </tr>
<tr> <td> 92 </td> <td>{Alpha, Delta, Gamma} </td> <td>{Alpha, Gamma, Delta}</td> </tr>
<tr> <td> 94 </td><td>if there is missing data </td> <td>if there are missing data </td> </tr>
<tr> <td> 99 </td> <td> index <em>i</em> in product<br> in equation 9.3 </td> <td> index should be <em>j</em> </td> </tr>
<tr> <td> 102 </td> <td> pioneering attempts </td> <td> pioneering attempt </td> </tr>
<tr> <td> 103 </td> <td> branch that applies to all characters </td> <td> character that applies to all branches </td> </tr>
<tr> <td> 105 </td> <td> so do the number of parameters </td> <td> so does the number of parameters </td> </tr>
<tr> <td> 105 </td> <td> this connected between </td> <td> this connection between </td> </tr>
<tr> <td> 106 </td> <td colspan=2> <div align=center> In equation 9.12, the expression is not correct.  The product over branches<br> should not be there at all.  The top and middle cases also need an extra factor of<br> <em>1/2</em>.  The assertions below the expression are still correct, however!  </div> </td> </tr>
<tr> <td> 112 </td> <td> and has presumable been </td> <td> and has presumably been </td> </tr>
<tr> <td> 112 </td> <td>(in Equation 9.16) &nbsp;&nbsp;&nbsp;&nbsp;<em>P<sub>1100</sub></em> </td> <td><em>P<sub>0011</sub></em> </td> </tr>
<tr> <td> 112 </td> <td>1100, but there is also the pattern 0011</td> <td>0011, but there is also the pattern 1100</td> </tr>
<tr> <td> 115 </td> <td> that there is probability </td> <td> there is probability </td> </tr>
<tr> <td> 117 </td> <td> as only parallel changes </td> <td> as parallel changes </td> </tr>
<tr> <td> 142 </td> <td> Sober chooses former </td> <td> Sober chooses the former </td></tr>
<tr> <td> 143 </td> <td>when the data is randomized</td> <td>when the data are randomized</td></tr>
<tr><td>144</td> <td>regard the experiment as repeatible</td><td>regard the experiment as repeatable</td></tr>
<tr> <td> 150 </td> <td> Equation 11.4 </td> <td> Before the big left parenthesis there<br> should be a term of <em>x<sub>ij,k</sub></em> </td> </tr>
<tr><td>151</td><td>(in equations 11.8 and 11.9)&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<b>D</b> </td><td>&nbsp;&nbsp;&nbsp;<b>d</b> </td></tr>
<tr><td>152</td><td>(in equations 11.11 and 11.12)&nbsp;&nbsp;&nbsp;<b>D</b> </td><td>&nbsp;&nbsp;&nbsp;<b>d</b> </td></tr>
<tr><td>152</td><td>11.10 (or 11.7)</td><td><b>X</b><sup>T</sup><b>X</b> (or <b>X</b><sup>T</sup><b> W X</b>)</td></tr>
<tr><td>152</td><td>11.12 (or 11.9)</td><td>11.9 (or 11.12)</td></tr>
<tr> <td> 161 </td> <td> indexclusteringclustering </td> <td> clustering </tr>
<tr> <td> 163 </td> <td> The two distance matrices<br> on this page are identical,<br> when they should not be.  </td> <td> The first should have no boldfacing, boxes<br> or asterisks.  It should be placed after<br> "branch length" in line 3.  The second<br> should be placed after "borders of the table".  </td> </tr>
<tr> <td> 169 </td> <td> indexclusteringclustered </td> <td> clustered </td> </tr>
<tr> <td> 173 </td> <td>extent to which the data departs</td> <td>extent to which the data depart</td> </tr>
<tr> <td> 180 </td> <td> it could still be fairly slow </td> <td> it could still be fairly fast </td> </tr>
<tr> <td> 181 </td> <td>likelihood methods infer from this data</td> <td>likelihood methods infer from these data </td> </tr>
<tr> <td> 183 </td> <td> {ABCDE&nbsp;|&nbsp;EFGHI} </td> <td> {ABCD&nbsp;|&nbsp;EFGHI} </td> </tr>
<tr> <td> 183 </td> <td> is discussed is their paper </td> <td> is discussed in their paper </td> </tr>
<tr> <td> 186 </td> <td>this data with a parsimony method</td> <td>these data with a parsimony method</td> </tr>
<tr> <td> 191 </td> <td> for the finding it </td> <td> for finding it </td> </tr>
<tr><td>191</td><td>and, if they are compatible finds </td><td>and, if they are compatible, finds </td></tr>
<tr> <td> 191 </td> <td> the same as the any </td> <td> the same as any </td> </tr>
<tr> <td>197</td> <td><table><tr><td></td><td align=center>2R-1</td></tr><tr><td>(equation 13.2)&nbsp;&nbsp;&nbsp;</td><td align=center>-------</td></tr><tr><td></td><td align=center>1+R</td></tr></table></td><td><table><tr><td>&nbsp;&nbsp;&nbsp;</td><td align=center>2R+1</td></tr><tr><td></td><td align=center>-------</td></tr><tr><td></td><td align=center>1+R</td></tr></table> </td> </tr>
<tr> <td> 204 </td> <td> <em>pi<sub>i</sub></em> in line 11 </td> <td> the <em>pi</em> should be the Greek letter &pi; instead </td> </tr>
<tr> <td> 204 </td> <td> first <em>n<sub>i</sub></em> in equation 13.12 </td> <td> <em>m<sub>i</sub></em> </td> </tr>
<tr><td>204</td><td>many C's at one end a branch</td><td>many Cs at one end of a branch</td></tr>
<tr><td>204</td><td>and many T's at the other</td><td>and many Ts at the other</td></tr>
<tr><td>204</td><td>the C's were in the ancestor</td><td>the Cs were in the ancestor</td></tr>
<tr><td>204</td><td>the T's in the descendant</td><td>the Ts in the descendant</td></tr>
<tr> <td> 206 </td> <td> eigenvectors lambda<sub>i</sub> and eigenvalues </td> <td> eigenvalues lambda<sub>i</sub> and eigenvectors </td> </tr>
<tr> <td> 206 </td><td>(in lower-right corner of first matrix<br> in equation 13.18) &nbsp;&nbsp;&nbsp; &pi;<sub>A</sub></td> <td>&pi;<sub>T</sub> </td> <td>  </td> </tr>
<tr> <td> 207 </td> <td> to give Table 13.5 </td> <td> to give Table 13.4 </td> </tr>
<tr> <td> 208 </td> <td> as in Table 13.5 </td> <td> as in Table 13.4 </td> </tr>
<tr> <td> 210 </td> <td>such as Table 13.5 </td> <td>such as Table 13.4</td> </tr>
<tr> <td> 213 </td> <td> approximate,du </td> <td> approximate, </td> </tr>
<tr><td>215</td><td><table><tr><td>(in equation 13.34)&nbsp;&nbsp;&nbsp;</td><td><table><tr><td><u>&nbsp;3&nbsp;</u></td></tr><tr><td>16</td></tr></table></td></tr></table></td><td align=center><table><tr><td><u>&nbsp;&nbsp;3&nbsp;&nbsp;</u></td></tr><tr><td>16n</td></tr></table></td></tr>
<tr> <td> 215 </td> <td> remarkable level </td> <td> remarkably level </td> </tr>
<tr> <td> 215 </td> <td>power of 1.0 </td> <td>power of 2.0</td> </tr>
<tr> <td> 215 </td> <td>w<sub>ij</sub> = 1/D<sub>ij</sub> </td> <td>w<sub>ij</sub> = 1/D<sub>ij</sub><sup>2</sup> </td> </tr>
<tr> <td>217</td> <td>x<sup>&alpha;-1</sup>e<sup>-x/&beta;</sup></td> <td>r<sup>&alpha;-1</sup>e<sup>-r/&beta;</sup></td></tr>
<tr> <td> 220 </td> <td><em>A</em>'s, <em>B</em>'s and <em>C</em>'s</td> <td><em>A</em>s, <em>B</em>s and <em>C</em>s</td> </tr>
<tr> <td> 220 </td> <td> such as exp(-brt) </td> <td> such as exp(-bt) </td> </tr>
<tr> <td> 220 </td> <td> E[e<sup>-brt</sup>] </td> <td> E<sub>r</sub>[e<sup>-brt</sup>] </td> </tr>
<tr> <td> 220 </td> <td> difference between<br> sequences (horizontal axis) </td> <td> difference between<br> sequences (vertical axis) </td> </tr>
<tr> <td> 225 </td> <td> is contributed by assignments </td> <td> contributed by the assignments </td> </tr>
<tr> <td> 226 </td> <td> (CGT, CGC) </td> <td> (AGC, AGT) </td> </tr>
<tr> <td> 232 </td> <td> (in equation 15.3)&nbsp;&nbsp;max[0,<em>l</em>-k]</td> <td>max[0,k-<em>l</em>]</td></tr>
<tr> <td> 232 </td> <td> The Kimura 2-parameter does this </td> <td> The Kimura 2-parameter model does this </td> </tr>
<tr> <td> 233 </td> <td>factor of 64</td> <td>factor of 16 </td></tr>
<tr> <td> 233 </td> <td>2&nbsp;x&nbsp;10<sup>8</sup> </td> <td>3&nbsp;x&nbsp;10<sup>5</sup> </td></tr>
<tr> <td>238 </td><td>(equation 15.22)<br>
<table><tr><td align=center>
   <table><tr><td></td><td>[1 - </td>
              <td>
         <table><tr><td align=center>1</td></tr>
                <tr><td align=center>-----</td></tr>
                <tr><td align=center>4<sup>r</sup>-1</td></tr>
         </table>
              </td>
              <td>(1-Q)]Q<sup>2</sup></td></tr>
   <tr><td>F&nbsp;&nbsp;=&nbsp;&nbsp;</td><td align=center colspan=3>------------------------</td></tr>
   <tr><td></td><td align=center colspan=3>2-Q</td></tr>
   </table>
</td></tr></table></td>
<td><table><tr><td></td><td align=center>Q<sup>2</sup></td></tr>
<tr><td>&nbsp;&nbsp;&nbsp;&nbsp;F&nbsp;=&nbsp;</td><td align=center>-------</td></tr>
<tr><td></td><td align=center>2-Q</td></tr></table> </td> </tr>
<tr><td>238</td> <td>we can use numerical methods to iterate<br> <em>Q</em> in equation (15.22) until the expected<br> fraction of restriction fragments
equals<br> the observed fraction <em>F</em>, </td> <td>we can note that equation (15.22)<br>
is a quadratic equation in <em>Q</em> and<br>solve it for <em>Q</em>,</td> </tr>
<tr> <td> 241 </td> <td> concerned with of genes </td> <td> concerned with coalescent trees of genes </td> </tr>
<tr> <td> 242 </td> <td>(in equation 15.26, twice)&nbsp;&nbsp;&nbsp;k=-infinity </td> <td>k=0 </td> </tr>
<tr> <td> 244 </td> <td> it does has some robustness </td> <td> it does have some robustness </td> </tr>
<tr> <td> 245 </td> <td>(in equation 15.28)<br>&nbsp;&nbsp;&nbsp;&mu; &nbsp; t&nbsp; &radic;(2 &pi;) </td> <td>&radic;(&mu; t) &radic;(2 &pi;) </td> </tr>
<tr><td>249</td> <td><table><tr><td></td><td align=center>Prob(D&nbsp;|&nbsp;H<sub>1</sub>)</td></tr><tr><td>(in equation 16.3)&nbsp;&nbsp;&nbsp;</td><td align=center>-----------------</td></tr><tr><td></td><td align=center>Prob(D&nbsp;|&nbsp;H<sub>2</sub>)</td></tr></table></td> <td><table><tr><td>&nbsp;&nbsp;&nbsp;</td><td align=center>Prob(H<sub>1</sub>&nbsp;|&nbsp;D)</td></tr><tr><td></td><td align=center>-----------------</td></tr><tr><td></td><td align=center>Prob(H<sub>2</sub>&nbsp;|&nbsp;D)</td></tr></table></td></tr>
<tr> <td>250</td> <td>Thus is does not show</td> <td>Thus it does not show</td> </tr>
<tr> <td> 253 </td> <td> in equation 16.11 </td> <td> in equation 16.10 </td> </tr>
<tr> <td> 253 </td> <td> Equation 16.11 can be </td> <td>Equation 16.10 can be </td> </tr>
<tr> <td>256</td> <td>(at end of equation 16.17)</td> <td> add these factors:&nbsp;&nbsp;L<sub>6</sub><sup>(i)</sup>(y) L<sub>8</sub><sup>(i)</sup>(z)</td> </tr>
<tr> <td>256</td> <td>(at end of equation 16.19)</td> <td> add these factors:&nbsp;&nbsp;L<sub>6</sub><sup>(i)</sup>(y) L<sub>8</sub><sup>(i)</sup>(z)</td> </tr>
<tr> <td> 257 </td> <td> [figure caption] around nodes 6 and 8 </td> <td> around branch 7 </td> </tr>
<tr> <td> 257-258 </td> <td> all occurences of node 6<br> or branch length t<sub>6</sub> </td> <td> ... should be node 7 or<br> branch length t<sub>7</sub> </td> </tr>
<tr> <td> 259 </td> <td> In equation 16.11 that would </td> <td> In equation 16.10 that would </td> </tr>
<tr> <td> 263 </td> <td> (in equation 16.28)&nbsp;&nbsp;&nbsp;r<sub>i<sub>j</sub></sub>, r<sub>i<sub>j+1</sub></sub>, ..., r<sub>i<sub>m</sub></sub></td> <td>&nbsp;&nbsp;&nbsp;r<sub>i<sub>j</sub></sub></td> </tr>
<tr> <td> 264 </td> <td colspan=2>in equation 16.29, before the summation sign add a factor &nbsp;&nbsp; Prob(D<sup>(j)</sup> | T, r<sub>i<sub>j</sub></sub>) <td> </tr>
<tr> <td> 265 </td> <td> sometimes too more flexibility </td> <td> sometimes more flexibility </td> </tr>
<tr><td>268</td><td>C0&nbsp;&nbsp;G0</td><td>G0&nbsp;&nbsp;C0</td></tr>
<tr><td>269</td><td>Using Gu's model,</td><td>Using a similar model,</td></tr>
<tr><td>271</td><td>number of character patters </td><td>number of character patterns </td></tr>
<tr><td>273</td><td>showed a particular interesting </td><td>showed a particularly interesting </td></tr>
<tr> <td> 276 </td> <td> <b>H</b><sub>3</sub> = <b>H</b><sub>2</sub>&otimes;<b>H</b><sub>2</sub> </td> <td> <b>H</b><sub>3</sub> = <b>H</b><sub>1</sub>&otimes;<b>H</b><sub>2</sub> </td> </tr>
<tr><td>277</td><td>branch in common.</td><td>branch in common, their path<br> lengths add to these sums.</td></tr>
<tr><td>279</td><td>r<sub>1</sub>+r<sub>2</sub>+r<sub>3</sub>+r<sub>4</sub> Note</td><td>r<sub>1</sub>+r<sub>2</sub>+r<sub>3</sub>+r<sub>4</sub>. Note</td><td></td></tr>
<tr><td>280</td><td>(equation 17.12)&nbsp;&nbsp;&nbsp;&nbsp;<b>q</b></td><td>&nbsp;&nbsp;&nbsp;<b>g</b></td></tr>
<tr><td>280</td><td>(equation 17.13)&nbsp;&nbsp;&nbsp;&nbsp;<b>q</b></td><td>&nbsp;&nbsp;&nbsp;<b>g</b></td></tr>
<tr><td>283</td><td>are denoted by q<sub>i</sub></td><td>&nbsp;&nbsp;&nbsp;g<sub>i</sub></td></tr>
<tr><td>283</td><td>(equation 17.15)&nbsp;&nbsp;&nbsp;&nbsp;q<sub>i</sub></td><td>&nbsp;&nbsp;&nbsp;g<sub>i</sub></td></tr>
<tr><td>284</td><td>those that have nonzero q<sub>i</sub></td><td>those that have nonzero g<sub>i</sub></td></tr>
<tr><td>284</td><td>minimizing C in equation 17.15</td><td>minimizing &Gamma;<sup>2</sup> in equation 17.15</td></tr>
<tr><td>289</td><td>Now some data is examined.</td><td>Now some data are examined.</td></tr>
<tr><td>289</td><td>less influence with 22 tosses</td><td>less influence with 44 tosses</td></tr>
<tr><td>289</td><td>data is half as probable as it is</td><td>data are half as probable as they are</td></tr>
<tr><td>291</td><td>Gomberg (1966)</td><td>Gomberg (1968)</td></tr>
<tr><td>293</td><td>drawn will be less that R</td><td>drawn will be less than R</td></tr>
<tr><td>300</td><td>followed by (Yang and<br> Rannala, 1997) by</td><td> also be followed (Yang and<br> Rannala, 1997) by </td></tr>
<tr><td>309-310</td><td colspan=2 align=center>the symbol for the vector <em>theta</em> should be boldface throughout</td></tr>
<tr> <td>310</td> <td>we look this up on</td> <td>we look this up in</td> </tr> 
<tr> <td>312</td> <td>(in legend of Figure 19.1) &nbsp;&nbsp;&nbsp;hightest likelihood</td> <td>highest likelihood</td> </tr> 
<tr> <td>314</td> <td>Thus we can find</td> <td>Thus, if we can find</td> </tr> 
<tr> <td> 315 </td> <td> branch lengths of evolutionary rates </td> <td> branch lengths or evolutionary rates </td> </tr>
<tr><td>316</td><td>penalize it by subtracting twice </td><td>penalize it by adding twice </td></tr>
<tr><td>319</td><td>one tree on that we see</td><td>one tree on which we see</td></tr>
<tr><td>328</td><td>whether a two data sets</td><td>whether two data sets</td></tr>
<tr><td>330</td><td>the closest fit to this data</td><td>the closest fit to these data</td></tr>
<tr><td>332</td><td>equal numbers changes in interior</td><td>equal numbers of changes in interior</td></tr>
<tr> <td>333</td> <td>(in equation 19.19)<br>
<table><tr><td>&nbsp;&nbsp;&nbsp;</td><td align=center>Prob(n<sub>1</sub>, n<sub>2</sub>, n<sub>3</sub> | p<sub>1</sub>, q<sub>1</sub>)</td></tr><tr><td></td><td align=center>-----------------------------</td> </tr><tr><td></td><td align=center>Prob(n<sub>1</sub>, n<sub>2</sub>, n<sub>3</sub> | p<sub>1</sub>, q<sub>1</sub>)</td></tr></table></td><td><table><tr><td>&nbsp;&nbsp;&nbsp;</td><td align=center>Prob(n<sub>1</sub>, n<sub>2</sub>, n<sub>3</sub> | p<sub>1</sub>, q<sub>1</sub>)</td></tr><tr><td></td><td align=center>-----------------------------</td> </tr><tr><td></td><td align=center>Prob(n<sub>1</sub>, n<sub>2</sub>, n<sub>3</sub> | p<sub>2</sub>, q<sub>2</sub>)</td></tr></table></td> </tr>
<tr> <td> 333</td> <td>&nbsp;&nbsp;p<sub>1</sub> = q<sub>1</sub> = 1/3</td> <td>&nbsp;&nbsp;p<sub>2</sub> = q<sub>2</sub> = 1/3</td></tr>
<tr> <td>333</td> <td>alternative values of <em>p<sub>2</sub></em> </td> <td>alternative values of <em>p<sub>1</sub></em>
<tr> <td> 340 </td> <td> between the delete-1/e jackknife and<br> the delete-1/e jackknife </td> <td> between the delete-1/e jackknife<br> and the bootstrap </td> </tr>
<tr> <td> <tr> <td> 347 </td> <td colspan=2> <div align=center>In Figure 20.4, the central curve is too dark,<br> and the rightmost curve has its left part too dark</div> </td> </tr>
<tr><td>350</td><td>triple bootsrap</td><td>triple bootstrap</td></tr>
<tr><td>356</td><td>rather that distance between </td><td>rather than distance between </td></tr>
<tr> <td> 364 </td> <td> their parsimony or likelihood<br> or likelihood scores </td> <td> their parsimony or likelihood scores </td></tr>
<tr> <td> 364 </td> <td>PHYLIP</td> <td> (Should be in regular Palatino font)</td> </tr>
<tr> <td> 365 </td> <td> Heading "An example" </td> <td> Should be a section head:<br> Roman font, bold face </td> </tr>
<tr><td>368</td><td>13514</td><td>13,514</td></tr>
<tr> <td> 375 </td> <td> 2415 </td> <td> 2,415 </td> </tr>
<tr> <td> 375 </td> <td>62,741</td> <td>(shift it half a digit to the right) </td> </tr>
<tr> <td> 376 </td><td>equal numbers of A's, C's, G's, and T's</td> <td>equal numbers of As, Cs, Gs, and Ts</td> </tr>
<tr><td>381</td><td>(in equation 22.17, first line)&nbsp;&nbsp;P<sub>xyyx</sub></td><td>&nbsp;&nbsp;P<sub>xyxz</sub><br>(terms in equation 22.17 have also been <br>arranged in a more logical order)</td></tr>
<tr><td>393</td><td>Gomberg (1966)</td><td>Gomberg (1968)</td></tr>
<tr> <td>393</td> <td>(in equation 23.4)<br>&nbsp;&nbsp;&nbsp;&sigma;<sup>2</sup> v<sub>12</sub>+&sigma;<sup>2</sup> v<sub>11</sub>+&sigma;<sup>2</sup> v<sub>7</sub> </td> <td>&nbsp;&nbsp;&sigma;<sup>2</sup> v<sub>12</sub>+&sigma;<sup>2</sup> v<sub>11</sub><br>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;+&sigma;<sup>2</sup>v<sub>10</sub>+&sigma;<sup>2</sup> v<sub>7</sub> </td> </tr>
<tr> <td> 394 </td> <td> v<sub>4</sub> in Figure 23.1 </td> <td> the <em>v</em> should be italic font </td> </tr>
<tr> <td> 395 </td> <td>species 1 by <em>x<sub>i1</sub></em></td> <td>species 1 by <em>x<sub>1i</sub></em></td></tr>
<tr> <td> 395 </td> <td>species 2 by <em>x<sub>i2</sub></em></td> <td>species 2 by <em>x<sub>2i</sub></em></td></tr>
<tr> <td>397</td> <td>(in equation 23.15)<br>
<table><tr><td><table><tr><td align=center>p</td></tr><tr><td align=center>&prod;</td></tr><tr><td align=center>i=1</td></tr></table></td><td>&sigma;<sub>i</sub><sup>-2p</sup></td>
<td>exp(&nbsp;-&nbsp;</td><td><table><tr><td align=center>1</td></tr><tr><td align=center>------------</td></tr><tr><td align=center>2 &sigma;<sub>i</sub><sup>2</sup></td></tr></table></td><td><table><tr><td align=center>p</td></tr><tr><td align=center>&sum;</td></tr><tr><td align=center>i=1</td></tr></table><td><table><tr><td align=center>(x<sub>1i</sub>-x<sub>2i</sub>)<sup>2</sup></td></tr><tr><td align=center>----------</td></td></tr><tr><td align=center>v<sub>1</sub>+v<sub>2</sub></sup> </td></td></tr></table><td> )</td></tr></table></td>
<td><table><tr><td>(</td><td><table><tr><td align=center>p</td></tr><tr><td align=center>&prod;</td></tr><tr><td align=center>i=1</td></tr></table></td><td>&sigma;<sub>i</sub><sup>-2</sup> )</td>
<td>exp(&nbsp;-&nbsp;</td><td><table><tr><td align=center>1</td></tr><tr><td align=center>---</td></tr><tr><td align=center>2</td></tr></table></td><td><table><tr><td align=center>p</td></tr><tr><td align=center>&sum;</td></tr><tr><td align=center>i=1</td></tr></table><td><table><tr><td align=center>(x<sub>1i</sub>-x<sub>2i</sub>)<sup>2</sup></td></tr><tr><td align=center>---------------------</td></td></tr><tr><td align=center>&sigma;<sub>i</sub><sup>2</sup>( v<sub>1</sub>+v<sub>2</sub></sup> ) </td></td></tr></table><td> )</td></tr></table></td></tr>
<tr> <td>398</td> <td>(in equation 23.16)<br>
<table><tr><td>-2p&nbsp;</td><td>&nbsp;ln(&sigma;<sub>i</sub>)&nbsp;-&nbsp;</td>
<td><table><tr><td align=center>1</td></tr><tr><td align=center>---</td></tr><tr><td align=center>2</td></tr></table></td><td>p&nbsp;ln(v<sub>1</sub>v<sub>2</sub>)&nbsp;-&nbsp;</td><td><table><tr><td align=center>1</td></tr><tr><td align=center>------------</td></tr><tr><td align=center>2&nbsp;&sigma;<sub>i</sub><sup>2</sup></td></tr></table></td><td><table><tr><td align=center>p</td></tr><tr><td align=center>&sum;</td></tr><tr><td align=center>i=1</td></tr></table><td><td><table><tr><td align=center>(x<sub>1i</sub>-x<sub>2i</sub>)<sup>2</sup></td></tr><tr><td align=center>-------------</td></td></tr><tr><td align=center>( v<sub>1</sub>+v<sub>2</sub></sup> ) </td></td></tr></table><td></tr></table></td>
<td><table><tr><td>-2&nbsp;</td><td><table><tr><td align=center>p</td></tr><tr><td align=center>&sum;</td></tr><tr><td align=center>i=1</td></tr></table><td><td>&nbsp;ln(&sigma;<sub>i</sub>)&nbsp;-&nbsp;</td>
<td><table><tr><td align=center>1</td></tr><tr><td align=center>---</td></tr><tr><td align=center>2</td></tr></table></td><td>p&nbsp;ln(v<sub>1</sub>v<sub>2</sub>)&nbsp;-&nbsp;</td><td><table><tr><td align=center>1</td></tr><tr><td align=center>---</td></tr><tr><td align=center>2</td></tr></table></td><td><table><tr><td align=center>p</td></tr><tr><td align=center>&sum;</td></tr><tr><td align=center>i=1</td></tr></table><td><td><table><tr><td align=center>(x<sub>1i</sub>-x<sub>2i</sub>)<sup>2</sup></td></tr><tr><td align=center>---------------------</td></td></tr><tr><td align=center>&sigma;<sub>i</sub><sup>2</sup>( v<sub>1</sub>+v<sub>2</sub></sup> ) </td></td></tr></table><td></tr></table></td>
</tr>
<tr> <td>402</td> <td>(in equation 23.33)&nbsp;&nbsp;&nbsp;&nbsp;<em>x</em> </td> <td>&nbsp;&nbsp;&nbsp;&nbsp;<b>x</b>&nbsp;&nbsp;&nbsp;(boldfaced)</td> </tr>
<tr> <td>413</td> <td>
<table><tr><td>(in equation 23.60)&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</td><td>&sum;</td><td>y<sub>ij</sub><sup>2</sup></td></tr>
<tr><td></td><td align=center>i</td><td></td></tr></table> </td> <td>
<table><tr><td>&nbsp;&nbsp;&nbsp;</td><td align=center>&sum;</td><td align=center>&sum;</td><td>y<sub>ij</sub><sup>2</sup></td></tr>
<tr><td></td><td align=center>i</td><td align=center>j</td><td></td></tr></table></td></tr>
<tr> <td> 417 </td> <td> frac1N </td> <td> <sup>1</sup><font size=+1>/</font><sub>N</sub> </td> </tr>
<tr> <td> 418 </td> <td>V<sub>A</sub>/(2N)</td> <td>V<sub>A</sub>/N</td> </tr>
<tr> <td> 418 </td> <td>(in equation 24.6)&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;(2NV<sub>M</sub>)/(2N)&nbsp;=&nbsp;V<sub>M</sub></td> <td>(2NV<sub>M</sub>)/N&nbsp;=&nbsp;2V<sub>M</sub></td> </tr>
<tr> <td> 418 </td> <td><em>N</em> times the variance of change</td> <td><em>2N</em> times the variance of change</td> </tr>
<tr> <td> 424 </td> <td> 1973b </td> <td> 1973a </td> </tr>
<tr> <td> 435 </td> <td>(in equation 25.1)&nbsp;&nbsp;&nbsp;&nbsp;Y<sub>2</sub>&nbsp;=&nbsp;y<sub>2</sub>/square root(0.75)</td><td>Y<sub>2</sub>&nbsp;=&nbsp;y<sub>2</sub>/square root(0.975)</td></td>
<tr> <td>439</td> <td>(in equation 25.2)&nbsp;&nbsp;&nbsp;&nbsp;<b>A</b>&otimes;<b>T</b> + <b>E</b>&otimes;<b>I</b> </td> <td> <b>T</b>&otimes;<b>A</b> + <b>I</b>&otimes;<b>E</b></td></tr>
<tr> <td>439</td> <td>matrix, <b>E</b>&otimes;<b>I</b> </td> <td> matrix, <b>I</b>&otimes;<b>E</b></td></tr>
<tr> <td>440</td> <td>(in equation 25.3)&nbsp;&nbsp;&nbsp;<b>I</b>&otimes;<b>&mu;</b>, A</b>&otimes;<b>T</b> + <b>E</b>&otimes;<b>I</b> </td> <td> <b>1</b>&otimes;<b>&mu;</b>, <b>T</b>&otimes;<b>A</b> + <b>I</b>&otimes;<b>E</b> </td>
<tr><td>457</td><td>have any number lineages </td><td>have any number of lineages </td></tr>
<tr> <td> 461 </td> <td> of equation 26.9 </td> <td> of equation 26.10 </td> </tr>
<tr><td>466</td><td>3 x 2 / 10<sup>6</sup> = 0.000006</td><td>3 x 2 /(4 x 10<sup>6</sup>) = 0.0000015</td></tr>
<tr><td>466</td><td>4 x 3 / 10<sup>6</sup> = 0.000012</td><td>4 x 3 / (4 x 10<sup>6</sup>) = 0.000003</td></tr>
<tr><td>466</td><td>0.000012+0.00000508</td><td>0.000003+0.00000608</td></tr>
<tr> <td> 467 </td> <td> ancestry of below </td> <td> ancestry below </td> </tr>
</tr> <tr> <td> 471 </td> <td> were one </td> <td> were once </td> </tr>
<tr> <td>472</td> <td><table><tr><td></td><td align=center>(n-1)</td></tr><td>(In equation 27.7)&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</td><td align=center>--------</td><tr></tr><td></td><td align=center>&Theta;</td><tr></tr></table></td>
<td><table><tr><td></td><td align=center>(n-1)</td></tr><td>&nbsp;&nbsp;-&nbsp;&nbsp;</td><td align=center>--------</td><tr></tr><td></td><td align=center>&Theta;</td><tr></tr></table></td></tr>
<tr> <td> 475 </td> <td> Prob(G&nbsp;|&nbsp;&Theta;<sub>0</sub>)<br> at start of line in middle of page </td> <td> Prob(G&nbsp;|&nbsp;&Theta;) </td>
<tr> <td>481</td> <td><table><tr><td>(in equation 27.22)&nbsp;&nbsp;&nbsp;&nbsp;</td><td><table><tr><td align=center>&sum;</td></tr><tr><td align=center>i</td></tr></table></td><td>c<sub>ijk</sub>(&Theta;<sub>0</sub>)</td><td><table><tr><td align=center>&sum;</td></tr><tr><td align=center>i</td></tr></table></td><td>a<sub>ijk</sub>(&Theta;)</td></tr></table></td><td><table><tr><td><table><tr><td><table><tr><td align=center>&sum;</td></tr><tr><td align=center>i</td></tr></table></td><td>a<sub>ijk</sub>(&Theta;<sub>0</sub>)</td></tr><tr><td colspan=2>-----------------------</td></tr><tr><td><table><tr><td align=center>&sum;</td></tr><tr><td align=center>i</td></tr></table></td><td>a<sub>ijk</sub>(&Theta;)</td></tr></table></td></tr></table></tr>
<tr> <td> 482 </td> <td> Heading "MCMC for a variety<br> of coalescent models" </td> <td> Should be a section heading:<br> Roman font, bold face </td></tr>
<tr><td>486</td><td>the data was represented</td><td>the data were represented</td></tr>
<tr><td>488</td><td>the copies from B and C are</td><td>the copies from A and C are</td></tr>
<tr><td>488</td><td>than either is to the copy from A</td><td>than either is to the copy from B</td></tr>
<tr> <td>494</td> <td>the <em>N - i</em> would actually</td> <td>the <em>N<sub>i</sub></em> would actually</td> </tr>
<tr> <td>498</td> <td>(in Figure caption):&nbsp;&nbsp;&nbsp;&nbsp;are shown by by the hash marks</td> <td>are shown by the hash marks</td> </tr>
<tr> <td>499</td> <td>an alignment of of position</td> <td>an alignment of position</td> </tr>
<tr><td>501</td><td>all the G's on the same side</td><td>all the Gs on the same side</td></tr>
<tr><td>501</td><td>the sequence in the two parts of the tree</td><td>the sequences in the two parts of the tree</td></tr>
<tr><td>504</td><td>Two T's are inserted to its right.</td><td>Two Ts are inserted to its right</td></tr>
<tr><td>504</td><td>Two T's are deleted.</td><td>Two Ts are deleted.</td></tr>
<tr><td>504</td><td>T's in the middle of the sequence</td><td>Ts in the middle of the sequence.</td></tr>
<tr><td>507</td><td>adjacent A's are deleted</td><td>adjacent As are deleted</td></tr>
<tr><td>507</td><td>turn out to be A's,</td><td>turn out to be As,</td></tr>
<tr><td>525</td><td>Heading: "A dismaying result"</td><td>Should be a subsection heading:<br>Italic font and not boldfaced</td></tr>
<tr> <td> 531 </td> <td>based on NNI's</td> <td>based on NNIs</td> </tr>
<tr> <td> 532 </td> <td> ranch lengths </td> <td> branch lengths </td> </tr>
<tr> <td> 544 </td> <td> parasite species of a species </td> <td> parasite species or a species </td> </tr>
<tr> <td> 548 </td> <td colspan=2> <div align=center>in Figure 32.1, for each large ellipse the<br> leftmost small ellipse within it is too dark</div> </td> </tr>
<tr> <td> 557 </td> <td>(in Figure 32.6)<br>
<table><tr> <td> <table> <tr> <td></td><td>-</td><td> <table> <tr><td align=center>4N<sub>e</sub></td></tr> <tr><td align=center>------</td></tr> <tr><td align=center>5x4</td></tr> </table> </td> <td>u</td> </tr> <tr><td>e</td><td></td><td></td></tr> </table> </td> <td>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;and&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</td> <td> <table><tr><td> <table> <tr><td</td></tr> <tr><td</td></tr> <tr><td</td></tr> <tr><td</td></tr> <tr><td</td></tr> <tr><td</td></tr> <tr><td</td></tr> <tr><td</td></tr> <tr><td</td></tr> <tr><td</td></tr> <tr><td align=center>1</td></tr> <tr><td align=center>-----</td></tr> <tr><td align=center>2N<sub>e</sub></td></tr> </table> </td> <td><table><tr><td></td><td>-</td><td><table><tr><td align=center>4N<sub>e</sub></td></tr><tr><td align=center>------</td></tr><tr><td align=center>7x6</td></tr></table></td><td>v</td><td></td></tr><tr><td>e</td><td></td><td></td><td></td><td>&nbsp;dv</td></tr></table> </td></tr></table> </td> </tr> </table>
<td>
<table><tr> <td> <table> <tr> <td></td><td>-</td><td> <table> <tr><td align=center>5x4</td></tr> <tr><td align=center>-----------</td></tr> <tr><td align=center>4N<sub>e</sub>&nbsp;&tau;</td></tr> </table> </td> <td>u</td> </tr> <tr><td>e</td><td></td><td></td></tr> </table> </td> <td>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;and&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</td> <td> <table><tr><td> <table> <tr><td</td></tr> <tr><td</td></tr> <tr><td</td></tr> <tr><td</td></tr> <tr><td</td></tr> <tr><td</td></tr> <tr><td</td></tr> <tr><td</td></tr> <tr><td</td></tr> <tr><td</td></tr> <tr><td align=center>1</td></tr> <tr><td align=center>-----</td></tr> <tr><td align=center>2N<sub>e</sub></td></tr> </table> </td> <td><table><tr><td></td><td>-</td><td><table><tr><td align=center>7x6</td></tr><tr><td align=center>-----------</td></tr><tr><td align=center>4N<sub>e</sub>&nbsp;&tau;</td></tr></table></td><td>v</td><td></td></tr><tr><td>e</td><td></td><td></td><td></td><td>&nbsp;dv</td></tr></table> </td></tr></table> </td> </tr> </table>
</td></tr>
<tr> <td> 557 </td> <td> exp[ </td> <td> exp[- </td> </tr>
<tr> <td> 561 </td> <td>probability that then urn</td> <td>probability that the urn</td> </tr>
<tr> <td> 578 </td> <td>subtrees of size 5, 7, and 1</td> <td>subtrees of size 8, 4, and 1</td> </tr>
<tr> <td> 578 </td> <td>these are species A-E, F-L, and M</td><td>these are species A-H, I-L, and M</td></tr>
<tr> <td> 578 </td> <td>angles 2.4166, 3.33833, and 0.24166</td> <td>angles 3.8666, 1.9333, and 0.48332</td> </tr>
<tr> <td>578</td> <td>Thus the five-species subtree has a branch length 10</td>
<td>Thus the eight-species subtree has a branch length 13</td></tr>
<tr> <td>578</td> <td>one with four species and one with one</td> <td>one with
five species and one with three</td></tr>
<tr> <td>578</td> <td>for the five-species subtree</td> <td>for the eight-species subtree</td> </tr>
<tr> <td>578</td> <td>branch of length 10</td> <td>branch of length 13</td> </tr>
<tr> <td>578</td> <td>Thus the five-species subtree</td> <td>Thus the eight-species subtree</td> </tr>
<tr> <td>578</td> <td>starts with an angle of 2.4166 radians</td> <td>starts with an angle of 3.8666 radians</td> </tr>
<tr> <td>578</td> <td>moves out a distance 10</td> <td>moves out a distance 13</td> </tr>
<tr> <td>578</td> <td>with four and one species</td> <td>with five and three species</td> </tr>
<tr> <td>578</td> <td>2.4166 radian sector</td> <td>3.8666 radian sector</td> </tr>
<tr> <td>578</td> <td>branch of length 10</td> <td>branch of length 13</td> </tr>
<tr> <td>578</td> <td>one of 1.9333 and the other of 0.48332</td> <td>one of 2.4166 and the other of 1.4500</td> </tr>
<tr> <td>578</td> <td>lengths 3 and 21</td> <td>lengths 10 and 13</td> </tr>
<tr> <td> 582 </td> <td colspan=2> <div align=center>dashed curves in Figure 34.6 need to<br> connect more accurately with dashed lines</div> </td> </tr>
<tr><td>612</td><td>Gomberg, D. 1966.</td><td>Gomberg, D. 1968.</td></tr>
<tr><td>619</td><td>Kishino, H., T. Miyata and</td><td>Kishino, H., T. Miyata, and</td></tr>
<tr><td>621</td><td>(in Lande 1976 reference) <b>31:</b> 442-446.</td><td><b>30:</b> 314-334.</td></tr>
<tr><td>623</td><td>(in reference for Lundy, 1985)&nbsp;&nbsp;198&nbsp;&nbsp;. </td><td>&nbsp;&nbsp;198. </td></tr>
<tr> <td>627</td> <td>(in Nielsen et al. 1998 reference):&nbsp;&nbsp;&nbsp;&nbsp;maximumlikelihood</td><td>maximum-likelihood</td></tr>
<tr> <td> 630 </td> <td> Purvis and Garland reference </td> <td> It is out of alphabetical order.<BR> It should follow Purdom et al.  </td> </tr>
<tr> <td> 641 </td> <td>(add reference after Wakeley 1998:)</td> <td>Wald, A.  1949. Note on the consistency of<br>
the maximum likelihood estimate.  <em>Annals<br>
of Mathematical Statistics</em> <b>20:</b> 595-601.</td> </tr>
<tr> <td> 644 </td> <td> posledobatel'nostyam. (in Russian) </td> <td> posledovatel'nostyam (in Russian).  </td> </tr>
<tr> <td>645 </td> <td> (after Angielczyk entry) </td> <td>Antonov, 631 </td> </tr>
<tr> <td>646</td> <td>Bousquet, 325 </td> <td>Bousquet, 325, 622 </td> </tr>
<tr> <td>648 </td> <td>Cucumel, 191, 527 </td> <td>Cucumel, 191, 527, 621 </td> </tr>
<tr> <td>651</td> <td>Hannenhalli, 516, 614 </td> <td>Hannenhalli, 516, 597, 614 </td> </tr>
<tr> <td>654</td> <td>Landrum, 95</td> <td>Landrum, 95, 605</td> </tr>
<tr> <td>655</td> <td>Margoliash, 131, 132, 134, 148, 222</td> <td>Margoliash, 131, 132, 134, 148, 222, 609</td> </tr>
<tr> <td>655</td> <td>Markowitz, 217, 268</td> <td>Markowitz, 217, 268, 609</td> </tr>
<tr> <td> 655 </td> <td>Meacham, 92, 95, 530, 578, 590, 624  </td> <td>Meacham, 92, 95, 530, 578, 590, 591, 605,<br> &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;624  </td> </tr>
<tr> <td> 660 </td> <td> Schoenberg, xx, 51 </td> <td> Schoenberg, xx, 40, 51 </td> </tr>
<tr> <td>660</td> <td>(after Solovyov entry)</td> <td>Soules, 597 </td> </tr>
<tr> <td> 663 </td> <td>Wald, 269-272</td> <td>Wald, 269-272, 641</td> </tr>
<tr> <td>664</td> <td>Zharkikh, 65, 66, 120, 213, 346, 349, 351,<br>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;631, 644</td> <td>Zharkikh, 65, 66, 120, 213, 346, 349, 351,<br>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;622, 631, 644</td> </tr>
</table>
<P>
These typos are corrected in the second printing.<br>
<P>
&nbsp;
<P>
I'd appreciate being told of any other typos that are discovered.<br>
There will be a chance to correct them at the next reprinting.
<P>
<table border=1>
<tr><td> Thanks for pointing out some of these to:</td></tr>
<tr><td>
<table align=center border=0>
<tr>
<td>Brian Anton</td>
<td>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</td>
<td>Simon Ho</td>
<td>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</td>
<td>Andrew Roger</td>
</tr>
<tr>
<td>David Bryant</td>
<td></td>
<td>Gangolf Jobb</td>
<td></td>
<td>Eric Rynes</td>
</tr>
<tr>
<td>Jose Clemente</td>
<td></td>
<td>Russ Lande</td>
<td></td>
<td>Dave Schruth</td>
<td></td>
</tr>
<tr>
<td>James Cotton</td>
<td></td>
<td>Dennis Lavrov</td>
<td></td>
<td>Nikolaus Schueler</td>
</tr>
<tr>
<td>Michael DiCuccio</td>
<td></td>
<td>Erick Matsen</td>
<td></td>
<td>Kelly Smith </td>
</tr>
<tr>
<td>Alexei Drummond</td>
<td></td>
<td>Peter Midford</td>
<td></td>
<td>Michael Sorenson</td>
<td></td>
</tr>
<tr>
<td>Casey Dunn</td>
<td></td>
<td>Shlomo Moran</td>
<td></td>
<td>John Trueman</td>
</tr>
<tr>
<td>Steve Evans</td>
<td></td>
<td>Peter Morcos</td>
<td></td>
<td>Jiaye Yu</td>
</tr>
<tr>
<td>Walter Fitch</td>
<td></td>
<td>Anders Gorm Pedersen</td>
<td></td>
<td>Jie Zha</td>
</tr>
<tr>
<td>Alex Griffing</td>
<td></td>
<td>Markus Riester</td>
<td></td>
<td></td>
</tr>
</table>
</td></tr>
<tr><td>
<p>
I am particularly grateful to John Trueman, Simon Ho, Peter Morcos, and Nik 
Schueler, who reported many of these typos.
</td></tr>
</table>
</DIV>
<P>
<a name="references"><H1>What got left out?</H1></a>
<P>
Of course the book omits all work after early August, 2003.  But some earlier
works were not discussed that should have been.  Particularly egregious
omissions include:
<UL>
<LI>reference to Nei and Kumar's (1999) book on molecular phylogenetics
<LI>reference to Paul Lewis's (2001) paper on a likelihood approach to
discretely-coded morphological characters.
<LI>reference to Semple and Steel's (2003) book <em>Phylogenetics</em>
<LI>mention of Hartigan's (<em>Biometrics</em>, 1973) paper which gives the
algorithm for counting number of parsimony changes in a multifurcating tree
with <em>n</em> unordered states.
<LI>Fredrik Ronquist, in his review of the book in <em>Science</em>,
points out that I did not cite two papers on speeding up parsimony
algorithms by Pablo Goloboff (1996 and 1998 in <em>Cladistics</em>),
and also a paper by Ward Wheeler (1995 in <em>Systematic Biology</em>)
on choosing alignment parameters by optimizing taxonomic congruence.
<LI> An explanation of Barry and Hartigan's 1987 computation of the likelihood
of a phylogeny using a general 12-parameter model, different for each branch of
the tree but the same across sites, should have
been provided, especially since their method is easy computationally.
There was a citation to their work in Chapter 13, but the methods for
the whole tree were not covered in Chapter 16 as they should have been.
<LI>The program <TT>MrBayes</TT> is quite widely used and cited, and should
have been included in the list of "Widely used phylogeny programs and packages".
<LI>Ziheng Yang's 1996 paper in <em>Journal of Molecular Evolution</em>
which gives a multi-locus likelihood method that estimates a separate rate
parameter for each locus should have been covered in the section of Chapter 30
on my suggestions for a likelihood approach to the Total Evidence controversy.
<LI>Although I have some coverage in chapters 13 and 14 of models of molecular
change that allow for nonindependence of sites in RNA and proteins, I should
have returned to the subject in Chapter 29 to cover the use of structure
in alignment.  This is actively used with ribosomal RNAs; many protein
researchers want to make use of structure, though the full integration of
structure into protein alignment is difficult and here the techniques are still
in their infancy.
</UL>
<P>
<a name="erroneous"><H1>What is incorrect?</H1></a>
<P>
As for incorrect arguments, there will be those who say most of the book
is incorrect. Obviously I don't agree.  The arguments I made that may be wrong
are:
<UL>
<LI>On page 234 in Chapter 15, where I discuss correcting for the
ascertainment of varying restriction sites, I promise that I give a more
general treatment of the subject in Chapter 16.  But you won't find it
there, as I forgot to do so.  Oops.  The more extensive
argument I would have given is to be found in the 1992 paper of mine that was
referred to on page 234.
<li> The statement on page 269 that the process in the covarion states changes
at a rate &delta; is wrong; Peter Morcos pointed out that it actually changes at
a rate  2 &kappa; &delta; / (1 + &kappa; &delta;).
<LI> In addition, on page 425, at the bottom, I say that "For the moment we
defer doing this until the next chapter."  I deferred it even longer than that,
as I never did return to the subject, even in the next chapter.
<LI> The "explanation" of the logic of the Akaike Information Criterion (AIC)
on page 316 must have made sense to me at the time I wrote it.  But I can't
make heads or tails of it now.
<LI> Some of the details of the importance sampling explanation of Griffiths and
Tavar&eacute;'s sampling method are incorrect, primarily the parts on page 481.
They would need substantial modification of the material on that page to be correct.  For a correct explanation
along the same lines, see the paper by Felsenstein et al. (1999) cited on page 477 where
the argument is similar, but correct. <a
href="http://projecteuclid.org/DPubS/Repository/1.0/Disseminate?handle=euclid.lnms/1215455552&view=body&content-type=pdf_1">Here</a> is a link to a free PDF of that paper.
The relevant material is in section 7 of the paper.
<LI>The description of Ward Wheeler's 1996 method of optimization aligment on
page 501 confuses it with his 1999 method of "fixed-states optimization".
Optimization alignment is an approximation, but not nearly as rough a one as
fixed-states optimization. (It was also reinvented, apparently independently,
by Lutzoni et al. in 2000 in Systematic Biology).  It delimits ambiguous
regions and uses a fixed-states approach on them, but not on the whole
sequence.  As such it cannot be dismissed summarily as I did on page 501.
<LI>Daniel Huson points out that my statement on page 178 that:
<BLOCKQUOTE>
We could check whether all quartets of species satisfy the FPM
condition, but this does not guarantee that there is one tree that
implies all of those individual FPM conditions.
</BLOCKQUOTE>
is blatantly incorrect.  If all possible FPM conditions are satisfied, it
has been known at least since Buneman's work that there must be a tree that
implies all of them.  A better statement would have been:
<BLOCKQUOTE>
We could check whether all quartets of species satisfy the FPM
condition; unless all possible quartets satisfy it, there is no
guarantee that there is one tree that implies all of the individual
FPM conditions.
</BLOCKQUOTE>
<LI> Apparently I accepted a myth.  In my introduction of Brownian motion
on page 391, I stated that Robert Brown "observed that pollen grains in
solution jiggled ceaselessly".    Actually, Brian Ford, in the paper I
cite there, argues that it was inclusions <em>within</em> the pollen grains
that were observed to jiggle as a result of Brownian motion.
David Wilkinson (in the letters section of the 10 March 2005 issue of <em>Nature</em>)
seems to argue that it was small particles <em>between</em> the pollen grains
that were seen to jiggle, but he tells me this was an editing mistake on the
part of <em>Nature</em>.  Pollen grains are apparently
too big to have observable Brownian motion themselves.
<LI> Daniel Shriner and co-workers have pointed out that the calculations of
numbers of unrooted unlabeled bifurcating tree shapes that I make in Table
3.5 are incorrect and overcount in most of the cases with an even number of
tips.  They note that a generating-function approach to this problem was given
by Annette Dobson in 1974 (<em>Journal of Applied Probability</em> <b>11:</b>
32-42, see pages 39-40).  Dobson's numbers show mine to be incorrect.
Interestingly, the method I describe in the text works -- I just didn't
implement it correctly when calculating the table.
The calculations for multifurcating unrooted tree shapes in that
table were also wrong for even numbers of species.  These errors are
corrected in the table of typos above.
<LI> Mike Steel has pointed out that the presentation on page 271 is unclear.
I should have explicitly noted that the tree T that is being evaluated is not
necessarily the true tree.  It will have expected pattern frequencies q<sub>i</sub>, while the true tree has expected pattern frequencies p<sub>i</sub>.
Given that, the proof presented is correct.
<LI> In F. James Rohlf's otherwise complimentary review of the book in
<em>Journal of Classification</em>, he points out that I 
misattributed the UPGMA method to Sokal and Michener's paper of
1957.  Oops. It was apparently first published by Rohlf in 1962 and by Peter Sneath
in the same year:
<ul>
<li>Rohlf, F. J. 1962. <em>A Numerical Taxonomic Study of the Genus Aedes (Diptera:
Culicidae) with Emphasis on the Congruence of Larval and Adult Classifications.</em> Ph.D. thesis, Department of Entomology, University of Kansas.
<li>Sneath, P. H. A. 1962.  The construction of taxonomic groups, pp. 289-332
 in <em>Microbial Classification,</em> eds. G. C. Ainsworth and P. H. A. Sneath. Cambridge University Press, Cambridge.
</ul>
<li> I found this one myself: on page 365, in the description of Swofford's
paired-sites <em>t</em>-test approach, I note that we can test sums of
differences with a test intended for means, "as these differ by a multiple
of <em>n</em> (the number of sites), and so do their variances."  That is
wrong: if the variance of a single difference is &sigma;<sup>2</sup>,
then the variance of the difference of means is <em>1/n</em> of this, and the
variance of the difference of sums is <em>n</em> times this.  A little
consideration will show that these variances differ by a factor of <em>n<sup>2</sup></em>,
not <em>n</em>.
<LI> The description of Pol and Siddall's 2001 simulations on page 121 could
be confusing.  I say that
<blockquote>
In their trees the long branches are in adjacent pairs.  This results
in parsimony doing better than likelihood.  As they lengthen the long
branches, both do worse, as expected.  At a certain length, parsimony falls
prey to a long branch attraction effect, causing likelihood to outperform it
beyond that length.
</blockquote>
There are really two effects of long branch attraction in their simulation,
one which brings adjacent branches together, and one, which occurs more when
branches are longer, which attracts more distant long branches.  This is left
fairly unclear by the current wording.
<li> Erick Matsen has pointed out that I described the Shao and Sokal (1990)
<em>B</em><sub>1</sub> tree shape statistic incorrectly on page 563.  It is computed by computing
for each interior node of the tree (except the root) the maximum of the number of branches
between it and each tip descended from it.  The reciprocals of these are then
summed.  For the tree in Figure 33.1 this gives  1/1 + 1/2 + 1/3 + 1/1 + 1/4 + 1/1 + 1/1 + 1/2 = 5.58333.
<li> David Bryant points out that my oh-so-obvious algorithm to do UPGMA
clustering in quadratic time (described on page 162) is actually wrong.
If the objects getting
clustered are nearest neighbors of many others, too many other distances
need to be updated.  (I suspect that this behavior will be rare in most
data sets and that the algorithm will on average work fairly well).
He notes that an elegant and simple quadratic algorithm for UPGMA was
found by Fionn Murtagh:<br>
<blockquote>
Murtagh, F. 1984. Complexities of hierarchic clustering algorithms: state
of the art.  <em>Computational Statistics Quarterly</em> <b>1:</b> 101-113.
</blockquote>
for which see scans of the paper at
<a href="https://thames.cs.rhul.ac.uk/~fionn/strule/old-articles/complexities/">this URL.</a>
<li> The chapter on the comparative method describes how the use of
contrasts enables us to find transformed character values that are
independent in their evolution.  It gives the impression that it is necessary
to have branch lengths <em>v<sub>i</sub></em> that are the variances that
are expected to accumulate in those characters.  For many purposes, it is
not necessary to know this much.  I should have explained in that chapter
that if we have branch lengths that are <em>proportional</em> to the variances,
we use those and the resulting transformations still achieve independence
and a common distribution,
and allow us to test covariation (correlation) among the characters.
<li> Andrew Roger points out that my equating clocklike trees with ultrametric
trees on page 161 is not quite right.  A tree could satisfy the ultrametric
conditions, in that there is a point in the tree equidistant from all tips,
but if it were not rooted at that point, it would not be clocklike.  Thus
the two are not quite the same.  Also, if the
rate of evolution speeds up (or slows
down) by the same amount in all lineages,
then the resulting tree will be
ultrametric but not clocklike.
<li> In the material in Chapter 16 on Hidden Markov Models for rate variation
along molecules, I explain the workings of the Backwards Algorithm for summing
up the likelihood in an efficient way.  It's fine except for the name I give
it.  The algorithm I explain is technically the Forwards Algorithm, not the
Backwards Algorithm.  The name is puzzling because it is working its way
backwards, not forwards, along the molecule.  (The names Forwards and Backwards are not good
ones because the way they differ is independent of which direction they are
used in  -- in this case perhaps we should have called it the
Forwards-Algorithm-Going-Backwards).
<li> Shlomo Moran points out that my statement on page 212 about the
LogDet distances for two successive branches (with an infinitely long
sequence) not adding up unless the intermediate base composition is
(1/4, 1/4, 1/4, 1/4) is wrong.  Formulas 13.29 through 13.32 can be used to
show that they <em>do</em> add up properly, and the results by Steel and
Lockhart cited on the next page, they are "tree additive" necessarily
imply that as well.  However it is still true, as in the numerical example
given in the top part of page 212, that the distances overestimate the
true branch lengths, even though they are tree-additive.
<LI> In Chapter 27 I show a likelihood curve for estimation of &Theta; in
Figure 27.2.  I say that it is from a simulated dataset with 50 tips and
500 sites.  I have now found the output file for that run (alas not yet the
data set itself) and find that the data set had 50 tips but only 200 sites.
<LI> In my discussion of restriction site models I discussed AFLP data, and
claimed that AFLP data would be similar to restriction fragment data in which
the length of the restriction site would be the sum of the lengths of the
recognition sequences of the two restriction sites used, plus the lengths of
the two primers used.  Liz Parks of North Carolina State University points out
that the lengths of the two primers should not be included because they are
molecules that are ligated on, and they are not previously present in the
organism and thus cannot mutate.
<LI> Peter Morcos points out that the Kronecker product structure of
the covariance matrix on page 420 would be clearer and more consistent with
the previous useage (pages 402-403) if the order of the matrices T and A
were reversed.  It can be done either way but his way is better.
<LI> On page 495, equation 28.6, the likelihood for multiple coalescent
trees in a species tree, is presented as if I derived it.  It was
actually presented by Wayne Maddison (1997) and I should have cited
him for that.
<li> Here's an embarrassing one.  In Chapter 10, on page 132, I show
the phylogeny that was published in Fitch and Margoliash's 1967 paper.  In
the caption I say that it was produced from distances based on cytochrome
sequences.  I also carefully include in the Fitch-Margoliash figure its
original caption from their paper.  But apparently I did not actually read
their caption closely.  It clearly says that the distances came from globin
sequences!
<li> On page 29, in the discussion leading to equation 3.2, I consider the
tree shapes ordered so that we consider the basal split of the tree into
subtrees whose sizes are 1:5, 2:4, then 3:3.   In the discussion after the
equation, and in the figure showing unrooted trees, they are presented in
the order 5:1, 4:2, then 3:3.  This of course is inconsistent.  For clarity,
one ordering or
the other should have been adopted.  It does not change the count of the
numbers of tree topologies as one could do that either way.
</UL>
<P>
<P>
<a name="2ndedition"><H1>What about a second edition?</H1></a>
<P>
There has been five further printings,
and there will be additional reprintings as necessary, with correction of
typos.  I don't think there is ever likely to be a full second edition.
Work on phylogenies is growing rapidly, and the second edition would have to
be twice the size.  Aside from whether people would buy
or read such a massive tome, there is the issue of who has the energy to
write it.  I don't.  Follow-up books probably either will have to be more
narrowly focussed on particular subfields, or will skim more lightly through
the literature, citing much less of it.
<P>
<H1>Downloads page</H1>
<P>
Those interested in downloading the data sets and trees that were used for the
examples in the book should look at the web page <A href="datasets.html">here</a>.
<P>
<H1>Reviews</H1>
<P>
A list of reviews of the book that have appeared, and some reactions to them,
will be found <a href="reviews.html">here<a/>.
<P>
<TABLE WIDTH=80%>
<TR><TD>
<DIV ALIGN=RIGHT>
Joe Felsenstein
</DIV>
</TD></TR>
</TABLE>
</BODY>
